diff --git a/ocs_ci/ocs/node.py b/ocs_ci/ocs/node.py
index 6f74f782..5b62b869 100644
--- a/ocs_ci/ocs/node.py
+++ b/ocs_ci/ocs/node.py
@@ -204,7 +204,7 @@ def drain_nodes(node_names):
     try:
         ocp.exec_oc_cmd(
             f"adm drain {node_names_str} --force=true --ignore-daemonsets "
-            f"--delete-local-data",
+            f"--delete-local-data --timeout=1810s",
             timeout=1800,
         )
     except TimeoutExpired:
