diff --git a/tests/functional/z_cluster/test_ceph_default_values_check.py b/tests/functional/z_cluster/test_ceph_default_values_check.py
index 2ecc6e60..2c0a4d0f 100644
--- a/tests/functional/z_cluster/test_ceph_default_values_check.py
+++ b/tests/functional/z_cluster/test_ceph_default_values_check.py
@@ -5,6 +5,7 @@ import pytest
 from ocs_ci.framework.pytest_customization.marks import (
     bugzilla,
     skipif_ocs_version,
+    skipif_ibm_power,
     brown_squad,
 )
 from ocs_ci.framework.testlib import (
@@ -103,6 +104,7 @@ class TestCephDefaultValuesCheck(ManageTest):
         config.DEPLOYMENT.get("ceph_debug"),
         reason="Ceph was configured with customized values by ocs-ci so there is point in validating its config values",
     )
+    @skipif_ibm_power
     def test_validate_ceph_config_values_in_rook_config_override(self):
         """
         Test case for comparing the cluster's config values of
diff --git a/tests/functional/z_cluster/test_must_gather.py b/tests/functional/z_cluster/test_must_gather.py
index 2685a8b2..f76620d0 100644
--- a/tests/functional/z_cluster/test_must_gather.py
+++ b/tests/functional/z_cluster/test_must_gather.py
@@ -2,7 +2,7 @@ import logging
 import pytest

 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ManageTest,
     tier1,
@@ -53,7 +53,13 @@ class TestMustGather(ManageTest):
                     skipif_hci_client,
                 ]
             ),
-            pytest.param(*["OTHERS"], marks=pytest.mark.polarion_id("OCS-1583")),
+            pytest.param(
+                *["OTHERS"],
+                marks=[
+                    pytest.mark.polarion_id("OCS-1583"),
+                    skipif_ibm_power,
+                ]
+            ),
         ],
     )
     @pytest.mark.skipif(
diff --git a/tests/functional/object/mcg/test_s3_with_java_sdk.py b/tests/functional/object/mcg/test_s3_with_java_sdk.py
index 729d3c04..775c5d0c 100644
--- a/tests/functional/object/mcg/test_s3_with_java_sdk.py
+++ b/tests/functional/object/mcg/test_s3_with_java_sdk.py
@@ -5,6 +5,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     bugzilla,
     skipif_ocs_version,
     skipif_disconnected_cluster,
+    skipif_ibm_power,
     skipif_proxy_cluster,
     tier1,
     red_squad,
@@ -21,6 +22,7 @@ logger = logging.getLogger(__name__)
 @runs_on_provider
 @skipif_ocs_version("<4.9")
 @skipif_disconnected_cluster
+@skipif_ibm_power
 @skipif_proxy_cluster
 class TestS3WithJavaSDK:
     @bugzilla("2064304")
diff --git a/tests/functional/object/mcg/test_noobaa_secret.py b/tests/functional/object/mcg/test_noobaa_secret.py
index d149e307..eb1af5a9 100644
--- a/tests/functional/object/mcg/test_noobaa_secret.py
+++ b/tests/functional/object/mcg/test_noobaa_secret.py
@@ -21,6 +21,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     bugzilla,
     skipif_ocs_version,
     skipif_disconnected_cluster,
+    skipif_ibm_power,
     red_squad,
     runs_on_provider,
     mcg,
@@ -185,6 +186,7 @@ class TestNoobaaSecrets:

     @bugzilla("2090956")
     @bugzilla("1992090")
+    @skipif_ibm_power
     @polarion_id("OCS-4468")
     def test_noobaa_secret_deletion_method2(self, teardown_factory, mcg_obj, cleanup):
         """
diff --git a/tests/functional/z_cluster/test_hugepages.py b/tests/functional/z_cluster/test_hugepages.py
index 29694183..c1b0005f 100644
--- a/tests/functional/z_cluster/test_hugepages.py
+++ b/tests/functional/z_cluster/test_hugepages.py
@@ -15,7 +15,7 @@ from ocs_ci.ocs.node import (
 from ocs_ci.ocs.resources.pod import (
     wait_for_pods_to_be_running,
 )
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     bugzilla,
     skipif_external_mode,
@@ -71,6 +71,7 @@ class TestHugePages(E2ETest):
         request.addfinalizer(finalizer)

     @skipif_external_mode
+    @skipif_ibm_power
     @pytest.mark.order(-3)
     def test_hugepages_post_odf_deployment(
         self,
diff --git a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
index 592f057e..7753a1f1 100644
--- a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
+++ b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
@@ -10,7 +10,7 @@ from ocs_ci.helpers.helpers import (
     get_last_log_time_date,
     check_osd_log_exist_on_rook_ceph_operator_pod,
 )
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ManageTest,
     tier2,
@@ -29,6 +29,7 @@ log = logging.getLogger(__name__)
 @bugzilla("1962821")
 @skipif_ocs_version("<4.8")
 @skipif_external_mode
+@skipif_ibm_power
 @pytest.mark.polarion_id("OCS-2581")
 class TestRookCephOperatorLogType(ManageTest):
     """
diff --git a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
index a2f706fb..a4e42259 100644
--- a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
+++ b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
@@ -1,6 +1,6 @@
 import logging

-from ocs_ci.framework.pytest_customization.marks import blue_squad
+from ocs_ci.framework.pytest_customization.marks import blue_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     polarion_id,
     skipif_aws_creds_are_missing,
@@ -24,6 +24,7 @@ log = logging.getLogger(__name__)
 @skipif_managed_service
 @skipif_disconnected_cluster
 @skipif_aws_creds_are_missing
+@skipif_ibm_power
 def test_noobaa_bucket_quota(measure_noobaa_exceed_bucket_quota, threading_lock):
     """
     Test that there are appropriate alerts when NooBaa Bucket Quota is reached.
diff --git a/tests/functional/object/mcg/test_mcg_resources_disruptions.py b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
index d4c5ee8a..92b15656 100644
--- a/tests/functional/object/mcg/test_mcg_resources_disruptions.py
+++ b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
@@ -18,6 +18,7 @@ from ocs_ci.framework.testlib import (
     runs_on_provider,
     mcg,
 )
+from ocs_ci.framework.pytest_customization.marks import skipif_ibm_power
 from ocs_ci.helpers import helpers
 from ocs_ci.helpers.helpers import wait_for_resource_state
 from ocs_ci.ocs import cluster, constants, ocp
@@ -130,7 +131,13 @@ class TestMCGResourcesDisruptions(MCGTest):
         argnames=["pod_to_drain"],
         argvalues=[
             pytest.param(*["noobaa_core"], marks=pytest.mark.polarion_id("OCS-2286")),
-            pytest.param(*["noobaa_db"], marks=pytest.mark.polarion_id("OCS-2287")),
+            pytest.param(
+                *["noobaa_db"],
+                marks=[
+                    pytest.mark.polarion_id("OCS-2287"),
+                    skipif_ibm_power,
+                ]
+            ),
             pytest.param(
                 *["noobaa_endpoint"], marks=pytest.mark.polarion_id("OCS-2288")
             ),
@@ -231,6 +238,7 @@ class TestMCGResourcesDisruptions(MCGTest):
     @pytest.mark.polarion_id("OCS-2513")
     @marks.bugzilla("1903573")
     @skipif_managed_service
+    @skipif_ibm_power
     @skipif_ocs_version("<4.7")
     def test_db_scc(self, teardown):
         """
diff --git a/tests/functional/object/mcg/test_host_node_failure.py b/tests/functional/object/mcg/test_host_node_failure.py
index c081a253..9769edfe 100644
--- a/tests/functional/object/mcg/test_host_node_failure.py
+++ b/tests/functional/object/mcg/test_host_node_failure.py
@@ -3,7 +3,7 @@ import pytest
 from concurrent.futures import ThreadPoolExecutor

 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import red_squad, mcg
+from ocs_ci.framework.pytest_customization.marks import red_squad, mcg, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     bugzilla,
     ignore_leftovers,
@@ -71,7 +71,10 @@ class TestNoobaaSTSHostNodeFailure(ManageTest):
             ),
             pytest.param(
                 *[constants.NOOBAA_DB_STATEFULSET, False],
-                marks=pytest.mark.polarion_id("OCS-2668"),
+                marks=[
+                    pytest.mark.polarion_id("OCS-2668"),
+                    skipif_ibm_power,
+                ]
             ),
             pytest.param(
                 *[constants.NOOBAA_CORE_STATEFULSET, True],
@@ -79,7 +82,10 @@ class TestNoobaaSTSHostNodeFailure(ManageTest):
             ),
             pytest.param(
                 *[constants.NOOBAA_DB_STATEFULSET, True],
-                marks=pytest.mark.polarion_id("OCS-2670"),
+                marks=[
+                    pytest.mark.polarion_id("OCS-2670"),
+                    skipif_ibm_power,
+                ]
             ),
         ],
     )
diff --git a/tests/functional/z_cluster/nodes/test_nodes_restart.py b/tests/functional/z_cluster/nodes/test_nodes_restart.py
index 5eb6f0b2..9bd95d6e 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_restart.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_restart.py
@@ -1,7 +1,7 @@
 import logging
 import pytest

-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     tier4a,
     tier4b,
@@ -140,6 +140,7 @@ class TestNodesRestart(ManageTest):
     )
     @skipif_ibm_cloud
     @skipif_vsphere_ipi
+    @skipif_ibm_power
     def test_pv_provisioning_under_degraded_state_stop_provisioner_pod_node(
         self,
         nodes,

