diff --git a/tests/manage/z_cluster/test_rook_ceph_log_rotate.py b/tests/manage/z_cluster/test_rook_ceph_log_rotate.py
index 73d22858..27750132 100644
--- a/tests/manage/z_cluster/test_rook_ceph_log_rotate.py
+++ b/tests/manage/z_cluster/test_rook_ceph_log_rotate.py
@@ -92,7 +92,7 @@ class TestRookCephLogRotate(ManageTest):
             else f"{self.podtype_id[pod_type][2]}{self.podtype_id[pod_type][1]}"
         )
         cnt_logs = len(re.findall(expected_string, output_cmd))
-        if cnt_logs != int(self.podtype_id[pod_type][3]) + 1:
+        if cnt_logs != int(self.podtype_id[pod_type][3]):
             log.info(output_cmd)
             log.error(
                 f"pod_type:{pod_type} cnt_logs_before_fill_log:"
