diff --git a/ocs_ci/ocs/resources/storage_cluster.py b/ocs_ci/ocs/resources/storage_cluster.py
index 2989ea74..67b0315e 100644
--- a/ocs_ci/ocs/resources/storage_cluster.py
+++ b/ocs_ci/ocs/resources/storage_cluster.py
@@ -583,7 +583,7 @@ def ocs_install_verification(
             deviceset_pvcs = list(set(deviceset_pvcs))
             if (
                 config.ENV_DATA.get("platform")
-                in [constants.BAREMETAL_PLATFORM, constants.HCI_BAREMETAL]
+                in [constants.BAREMETAL_PLATFORM, constants.HCI_BAREMETAL, constants.IBM_POWER_PLATFORM]
                 or config.ENV_DATA.get("platform") == constants.AWS_PLATFORM
             ):
                 deviceset_pvcs = [

