diff --git a/ocs_ci/deployment/helpers/lso_helpers.py b/ocs_ci/deployment/helpers/lso_helpers.py
index 6dbcaa918..17f37db8f 100644
--- a/ocs_ci/deployment/helpers/lso_helpers.py
+++ b/ocs_ci/deployment/helpers/lso_helpers.py
@@ -18,6 +18,7 @@ from ocs_ci.ocs.node import (
     get_node_objs,
 )
 from ocs_ci.utility import templating, version
+from ocs_ci.ocs.utils import label_pod_security_admission
 from ocs_ci.utility.deployment import get_ocp_ga_version
 from ocs_ci.utility.localstorage import get_lso_channel
 from ocs_ci.utility.retry import retry
@@ -123,6 +124,7 @@ def setup_local_storage(storageclass):
     if platform == constants.RHV_PLATFORM:
         add_disk_for_rhv_platform()
 
+    label_pod_security_admission(lso_namespace)
     if (ocp_version >= version.VERSION_4_6) and (ocs_version >= version.VERSION_4_6):
         # Pull local volume discovery yaml data
         logger.info("Pulling LocalVolumeDiscovery CR data from yaml")
