diff --git a/ocs_ci/deployment/helpers/lso_helpers.py b/ocs_ci/deployment/helpers/lso_helpers.py
index 6bd5ff080..316603132 100644
--- a/ocs_ci/deployment/helpers/lso_helpers.py
+++ b/ocs_ci/deployment/helpers/lso_helpers.py
@@ -21,6 +21,7 @@ from ocs_ci.ocs.node import (
     get_master_nodes,
 )
 from ocs_ci.utility import templating, version
+from ocs_ci.ocs.utils import label_pod_security_admission
 from ocs_ci.utility.deployment import get_ocp_ga_version
 from ocs_ci.utility.operators import LocalStorageOperator
 from ocs_ci.utility.localstorage import get_lso_channel
@@ -62,7 +63,8 @@ def setup_local_storage(storageclass):
     lso_type = config.DEPLOYMENT.get("type")
 
     add_disks_lso()
-
+    lso_namespace = config.ENV_DATA["local_storage_namespace"]
+    label_pod_security_admission(lso_namespace)
     if (ocp_version >= version.VERSION_4_6) and (ocs_version >= version.VERSION_4_6):
         # Pull local volume discovery yaml data
         logger.info("Pulling LocalVolumeDiscovery CR data from yaml")
