diff --git a/ocs_ci/ocs/ripsaw.py b/ocs_ci/ocs/ripsaw.py
index 032e541f..2e8179b1 100644
--- a/ocs_ci/ocs/ripsaw.py
+++ b/ocs_ci/ocs/ripsaw.py
@@ -42,9 +42,9 @@ class RipSaw(object):
         """
         self.args = kwargs
         self.repo = self.args.get(
-            "repo", "https://github.com/cloud-bulldozer/benchmark-operator"
+            "repo", "https://github.com/ocp-power-automation/benchmark-operator"
         )
-        self.branch = self.args.get("branch", "master")
+        self.branch = self.args.get("branch", "dev-ppc64le")
         self.namespace = self.args.get("namespace", RIPSAW_NAMESPACE)
         self.pgsql_is_setup = False
         self.ocp = OCP()
diff --git a/tests/e2e/performance/test_fio_benchmark.py b/tests/e2e/performance/test_fio_benchmark.py
index 1e1527d6..0cdaa0ee 100644
--- a/tests/e2e/performance/test_fio_benchmark.py
+++ b/tests/e2e/performance/test_fio_benchmark.py
@@ -343,11 +343,12 @@ class TestFIOBenchmark(PASTest):
         log.info(f"list of all PVs :{pvs_list}")
 
         for line in pvs_list:
-            pv, ns = line.split(" ")
-            pv = pv.replace("'", "")
-            if ns == constants.RIPSAW_NAMESPACE:
-                log.info(f"Going to delete {pv}")
-                run_command(f"oc delete pv {pv}")
+            if "<no value>" not in line:
+                pv, ns = line.split(" ")
+                pv = pv.replace("'", "")
+                if ns == constants.RIPSAW_NAMESPACE:
+                    log.info(f"Going to delete {pv}")
+                    run_command(f"oc delete pv {pv}")
 
     def run(self):
         """
