diff --git a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
index 2b741b6a3..cbadbeab6 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
@@ -53,6 +53,9 @@ from ocs_ci.helpers.helpers import (
 )
 from ocs_ci.helpers import helpers
 
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
 
 log = logging.getLogger(__name__)
 
@@ -124,6 +127,7 @@ class TestNodesMaintenance(ManageTest):
     @tier1
     @skipif_managed_service
     @skipif_hci_provider_and_client
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["node_type"],
         argvalues=[
@@ -195,6 +199,7 @@ class TestNodesMaintenance(ManageTest):
     @skipif_bm
     @skipif_managed_service
     @skipif_hci_provider_and_client
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["node_type"],
         argvalues=[
@@ -474,6 +479,7 @@ class TestNodesMaintenance(ManageTest):
     @skipif_managed_service
     @skipif_hci_provider_and_client
     @skipif_more_than_three_workers
+    @skipif_ibm_power
     @pytest.mark.polarion_id("OCS-2524")
     @tier4a
     def test_pdb_check_simultaneous_node_drains(
