diff --git a/tests/functional/z_cluster/test_ceph_default_values_check.py b/tests/functional/z_cluster/test_ceph_default_values_check.py
index 3333e7470..9a524f2d7 100644
--- a/tests/functional/z_cluster/test_ceph_default_values_check.py
+++ b/tests/functional/z_cluster/test_ceph_default_values_check.py
@@ -6,6 +6,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     bugzilla,
     skipif_ocs_version,
     brown_squad,
+    skipif_ibm_power,
 )
 from ocs_ci.framework.testlib import (
     ManageTest,
@@ -34,6 +35,7 @@ log = logging.getLogger(__name__)
 @brown_squad
 @tier1
 @skipif_external_mode
+@skipif_ibm_power
 @pytest.mark.polarion_id("OCS-2231")
 @bugzilla("1908414")
 class TestCephDefaultValuesCheck(ManageTest):
diff --git a/tests/functional/z_cluster/test_must_gather.py b/tests/functional/z_cluster/test_must_gather.py
index c49d46ef6..b2451379c 100644
--- a/tests/functional/z_cluster/test_must_gather.py
+++ b/tests/functional/z_cluster/test_must_gather.py
@@ -2,7 +2,7 @@ import logging
 import pytest
 
 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ManageTest,
     tier1,
@@ -57,7 +57,7 @@ class TestMustGather(ManageTest):
                 ]
             ),
             pytest.param(
-                *["OTHERS"], marks=[tier2, pytest.mark.polarion_id("OCS-1583")]
+                *["OTHERS"], marks=[tier2, pytest.mark.polarion_id("OCS-1583"), skipif_ibm_power]
             ),
         ],
     )
diff --git a/tests/functional/object/mcg/test_s3_with_java_sdk.py b/tests/functional/object/mcg/test_s3_with_java_sdk.py
index 729d3c046..d1c9a22ff 100644
--- a/tests/functional/object/mcg/test_s3_with_java_sdk.py
+++ b/tests/functional/object/mcg/test_s3_with_java_sdk.py
@@ -10,6 +10,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     red_squad,
     runs_on_provider,
     mcg,
+    skipif_ibm_power,
 )
 from ocs_ci.ocs.bucket_utils import upload_objects_with_javasdk
 
@@ -22,6 +23,7 @@ logger = logging.getLogger(__name__)
 @skipif_ocs_version("<4.9")
 @skipif_disconnected_cluster
 @skipif_proxy_cluster
+@skipif_ibm_power
 class TestS3WithJavaSDK:
     @bugzilla("2064304")
     @pytest.mark.parametrize(
diff --git a/tests/functional/object/mcg/test_noobaa_secret.py b/tests/functional/object/mcg/test_noobaa_secret.py
index b4996b010..2b0d06558 100644
--- a/tests/functional/object/mcg/test_noobaa_secret.py
+++ b/tests/functional/object/mcg/test_noobaa_secret.py
@@ -19,6 +19,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     runs_on_provider,
     mcg,
     post_upgrade,
+    skipif_ibm_power,
 )
 from ocs_ci.ocs.exceptions import CommandFailed
 from ocs_ci.utility.aws import update_config_from_s3
@@ -68,6 +69,7 @@ def cleanup(request):
 @runs_on_provider
 @skipif_ocs_version("<4.11")
 @skipif_disconnected_cluster
+@skipif_ibm_power
 class TestNoobaaSecrets:
     @bugzilla("1992090")
     @polarion_id("OCS-4466")
diff --git a/tests/functional/z_cluster/test_hugepages.py b/tests/functional/z_cluster/test_hugepages.py
index 5fabde8b8..7056f9c80 100644
--- a/tests/functional/z_cluster/test_hugepages.py
+++ b/tests/functional/z_cluster/test_hugepages.py
@@ -15,7 +15,7 @@ from ocs_ci.ocs.node import (
 from ocs_ci.ocs.resources.pod import (
     wait_for_pods_to_be_running,
 )
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     bugzilla,
     skipif_external_mode,
@@ -35,6 +35,7 @@ log = logging.getLogger(__name__)
 @bugzilla("2001933")
 @pytest.mark.polarion_id("OCS-2754")
 @ignore_leftovers
+@skipif_ibm_power
 class TestHugePages(E2ETest):
     """
     Enable huge pages post ODF installation
diff --git a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
index 592f057e3..7753a1f1f 100644
--- a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
+++ b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
@@ -10,7 +10,7 @@ from ocs_ci.helpers.helpers import (
     get_last_log_time_date,
     check_osd_log_exist_on_rook_ceph_operator_pod,
 )
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ManageTest,
     tier2,
@@ -29,6 +29,7 @@ log = logging.getLogger(__name__)
 @bugzilla("1962821")
 @skipif_ocs_version("<4.8")
 @skipif_external_mode
+@skipif_ibm_power
 @pytest.mark.polarion_id("OCS-2581")
 class TestRookCephOperatorLogType(ManageTest):
     """
diff --git a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
index a2f706fbe..5472cf4b0 100644
--- a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
+++ b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
@@ -1,6 +1,6 @@
 import logging
 
-from ocs_ci.framework.pytest_customization.marks import blue_squad
+from ocs_ci.framework.pytest_customization.marks import blue_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     polarion_id,
     skipif_aws_creds_are_missing,
@@ -20,6 +20,7 @@ log = logging.getLogger(__name__)
 @mcg
 @blue_squad
 @tier2
+@skipif_ibm_power
 @polarion_id("OCS-1254")
 @skipif_managed_service
 @skipif_disconnected_cluster
diff --git a/tests/functional/object/mcg/test_mcg_resources_disruptions.py b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
index d4c5ee8ac..1be256feb 100644
--- a/tests/functional/object/mcg/test_mcg_resources_disruptions.py
+++ b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
@@ -4,6 +4,7 @@ from ocs_ci.framework import config
 import pytest
 
 from ocs_ci.framework.pytest_customization import marks
+from ocs_ci.framework.pytest_customization.marks import skipif_ibm_power
 from ocs_ci.framework.testlib import (
     MCGTest,
     ignore_leftovers,
@@ -62,7 +63,7 @@ class TestMCGResourcesDisruptions(MCGTest):
     @pytest.mark.parametrize(
         argnames=["resource_to_delete"],
         argvalues=[
-            pytest.param(*["noobaa_core"], marks=pytest.mark.polarion_id("OCS-2232")),
+            pytest.param(*["noobaa_core"], marks=[pytest.mark.polarion_id("OCS-2232"), skipif_ibm_power,]),
             pytest.param(*["noobaa_db"], marks=pytest.mark.polarion_id("OCS-2233")),
         ],
     )
@@ -92,6 +93,7 @@ class TestMCGResourcesDisruptions(MCGTest):
     @skipif_ocs_version("<4.5")
     @on_prem_platform_required
     @skipif_external_mode
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["scale_down_to"],
         argvalues=[
diff --git a/tests/functional/object/mcg/test_host_node_failure.py b/tests/functional/object/mcg/test_host_node_failure.py
index c081a2532..7077df6f3 100644
--- a/tests/functional/object/mcg/test_host_node_failure.py
+++ b/tests/functional/object/mcg/test_host_node_failure.py
@@ -3,7 +3,7 @@ import pytest
 from concurrent.futures import ThreadPoolExecutor
 
 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import red_squad, mcg
+from ocs_ci.framework.pytest_customization.marks import red_squad, mcg, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     bugzilla,
     ignore_leftovers,
@@ -38,6 +38,7 @@ log = logging.getLogger(__name__)
 @bugzilla("1853638")
 @ignore_leftovers
 @skipif_vsphere_ipi
+@skipif_ibm_power
 @skipif_ocs_version("<4.9")
 class TestNoobaaSTSHostNodeFailure(ManageTest):
     """
diff --git a/tests/functional/z_cluster/nodes/test_nodes_restart.py b/tests/functional/z_cluster/nodes/test_nodes_restart.py
index 70172f4a7..0decbd8d1 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_restart.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_restart.py
@@ -1,7 +1,7 @@
 import logging
 import pytest
 
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     tier4a,
     tier4b,
@@ -37,6 +37,7 @@ logger = logging.getLogger(__name__)
 @ignore_leftovers
 @skipif_managed_service
 @skipif_hci_provider_and_client
+@skipif_ibm_power 
 class TestNodesRestart(ManageTest):
     """
     Test ungraceful cluster shutdown
