diff --git a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
index b1123a2c..f823170b 100644
--- a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
@@ -2,6 +2,10 @@ import logging
 from concurrent.futures import ThreadPoolExecutor
 import pytest
 
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
+
 from ocs_ci.ocs.cluster import is_flexible_scaling_enabled
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import pod as pod_helpers
@@ -41,6 +45,7 @@ logger = logging.getLogger(__name__)
 @ignore_leftovers
 @tier2
 @skipif_bm
+@skipif_ibm_power
 @skipif_external_mode
 @skipif_managed_service
 class TestAddCapacity(ManageTest):
diff --git a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
index ed9a34d7..357259b9 100644
--- a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
@@ -1,6 +1,10 @@
 import pytest
 import logging
 
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
+
 from ocs_ci.framework.pytest_customization.marks import brown_squad
 from ocs_ci.framework.testlib import (
     ignore_leftovers,
@@ -37,6 +41,7 @@ logger = logging.getLogger(__name__)
 @ignore_leftovers
 @tier4b
 @skipif_managed_service
+@skipif_ibm_power
 class TestAddCapacityNodeRestart(ManageTest):
     """
     Test add capacity when one of the worker nodes got restart
diff --git a/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py b/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py
index ce51eeb7..5d0ed3cb 100644
--- a/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_delete_pod.py
@@ -6,6 +6,7 @@ from concurrent.futures import ThreadPoolExecutor
 from ocs_ci.framework.testlib import ignore_leftovers, tier4c, skipif_managed_service
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_external_mode,
+    skipif_ibm_power,
     brown_squad,
 )
 from ocs_ci.framework import config
@@ -29,6 +30,7 @@ logger = logging.getLogger(__name__)
 @skipif_external_mode
 @ignore_leftovers
 @tier4c
+@skipif_ibm_power
 class TestAddCapacityWithResourceDelete:
     """
     Test add capacity when one of the resources gets deleted
diff --git a/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py b/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py
index d8463314..1955add0 100644
--- a/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_node_expansion.py
@@ -6,6 +6,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_flexy_deployment,
     skipif_ibm_flash,
     skipif_managed_service,
+    skipif_ibm_power,
     brown_squad,
 )
 
@@ -19,6 +20,7 @@ logger = logging.getLogger(__name__)
 @skipif_ibm_flash
 @ignore_leftovers
 @tier1
+@skipif_ibm_power
 class TestAddNode(ManageTest):
     """
     Automates adding worker nodes to the cluster while IOs
diff --git a/tests/manage/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py b/tests/manage/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
index d498bc8b..1a857ebd 100644
--- a/tests/manage/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
@@ -5,6 +5,7 @@ from ocs_ci.ocs import constants
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_flexy_deployment,
     skipif_managed_service,
+    skipif_ibm_power,
     skipif_multus_enabled,
     brown_squad,
 )
@@ -20,6 +21,7 @@ logger = logging.getLogger(__name__)
 @skipif_flexy_deployment
 @skipif_managed_service
 @skipif_multus_enabled
+@skipif_ibm_power
 @tier2
 @pytest.mark.polarion_id("OCS-2490")
 @pytest.mark.bugzilla("1794389")
diff --git a/tests/manage/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py b/tests/manage/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
index 5c7e695c..8e0fc57f 100644
--- a/tests/manage/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
@@ -4,6 +4,7 @@ import pytest
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_managed_service,
     skipif_bm,
+    skipif_ibm_power,
     brown_squad,
 )
 from ocs_ci.ocs.node import drain_nodes, schedule_nodes, is_node_rack_or_zone_exist
@@ -38,6 +39,7 @@ logger = logging.getLogger(__name__)
 @bugzilla("1898808")
 @skipif_bm
 @skipif_external_mode
+@skipif_ibm_power
 @skipif_managed_service
 @pytest.mark.polarion_id("OCS-2594")
 class TestAddNodeCrashCollector(ManageTest):
diff --git a/tests/manage/z_cluster/nodes/test_node_replacement_proactive.py b/tests/manage/z_cluster/nodes/test_node_replacement_proactive.py
index 6e82274a..b74d7b72 100644
--- a/tests/manage/z_cluster/nodes/test_node_replacement_proactive.py
+++ b/tests/manage/z_cluster/nodes/test_node_replacement_proactive.py
@@ -20,6 +20,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     bugzilla,
     skipif_external_mode,
     skipif_ms_consumer,
+    skipif_ibm_power,
     brown_squad,
 )
 from ocs_ci.helpers.helpers import verify_storagecluster_nodetopology
@@ -180,6 +181,7 @@ def delete_and_create_osd_node(osd_node_name):
 @skipif_managed_service
 @skipif_bmpsi
 @skipif_external_mode
+@skipif_ibm_power
 class TestNodeReplacementWithIO(ManageTest):
     """
     Knip-894 Node replacement proactive with IO
@@ -258,6 +260,7 @@ class TestNodeReplacementWithIO(ManageTest):
 @skipif_bmpsi
 @skipif_external_mode
 @skipif_ms_consumer
+@skipif_ibm_power
 class TestNodeReplacement(ManageTest):
     """
     Knip-894 Node replacement proactive
@@ -305,6 +308,7 @@ class TestNodeReplacement(ManageTest):
 @pytest.mark.polarion_id("OCS-2535")
 @skipif_external_mode
 @skipif_managed_service
+@skipif_ibm_power
 class TestNodeReplacementTwice(ManageTest):
     """
     Node replacement twice:
