diff --git a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
index 25cdc7740..8199d0174 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
@@ -39,6 +39,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     brown_squad,
     skipif_hci_provider,
     skipif_rosa_hcp,
+    skipif_ibm_power,
     skipif_compact_mode,
     runs_on_provider,
 )
@@ -151,6 +152,7 @@ class TestNodesMaintenance(ManageTest):
 
     @tier1
     @skipif_managed_service
+    @skipif_ibm_power
     @skipif_hci_provider
     @pytest.mark.parametrize(
         argnames=["node_type"],
@@ -229,6 +231,7 @@ class TestNodesMaintenance(ManageTest):
 
     @tier4a
     @skipif_bm
+    @skipif_ibm_power
     @skipif_managed_service
     @pytest.mark.parametrize(
         argnames=["node_type"],
@@ -311,6 +314,7 @@ class TestNodesMaintenance(ManageTest):
         self.sanity_helpers.delete_resources()
 
     @tier3
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["nodes_type"],
         argvalues=[
@@ -347,6 +351,7 @@ class TestNodesMaintenance(ManageTest):
 
     @tier2
     @pytest.mark.polarion_id("OCS-1274")
+    @skipif_ibm_power
     @skipif_compact_mode
     def test_2_nodes_different_types(
         self, pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory
@@ -510,6 +515,7 @@ class TestNodesMaintenance(ManageTest):
 
     @skipif_managed_service
     @skipif_more_than_three_workers
+    @skipif_ibm_power
     @pytest.mark.polarion_id("OCS-2524")
     @tier4a
     def test_pdb_check_simultaneous_node_drains(
