diff --git a/ocs_ci/ocs/resources/storage_cluster.py b/ocs_ci/ocs/resources/storage_cluster.py
index 43da7685..06a044b3 100644
--- a/ocs_ci/ocs/resources/storage_cluster.py
+++ b/ocs_ci/ocs/resources/storage_cluster.py
@@ -410,7 +410,7 @@ def ocs_install_verification(
             deviceset_pvcs = [osd.get_node() for osd in get_osd_pods()]
             # removes duplicate hostname
             deviceset_pvcs = list(set(deviceset_pvcs))
-            if config.ENV_DATA.get("platform") == constants.BAREMETAL_PLATFORM or (
+            if config.ENV_DATA.get("platform") == constants.BAREMETAL_PLATFORM or constants.IBM_POWER_PLATFORM or (
                 config.ENV_DATA.get("flexy_deployment")
                 and config.ENV_DATA.get("platform") == constants.AWS_PLATFORM
             ):
