diff --git a/tests/cross_functional/scale/test_scale_osds_fill_75%_reboot_workers.py b/tests/cross_functional/scale/test_scale_osds_fill_75%_reboot_workers.py
index 151f08fe9..58537b535 100644
--- a/tests/cross_functional/scale/test_scale_osds_fill_75%_reboot_workers.py
+++ b/tests/cross_functional/scale/test_scale_osds_fill_75%_reboot_workers.py
@@ -21,6 +21,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     orange_squad,
     skipif_external_mode,
     skipif_aws_i3,
+    skipif_ibm_power,
 )
 
 
@@ -31,6 +32,7 @@ logger = logging.getLogger(__name__)
 @scale
 @ignore_leftovers
 @skipif_external_mode
+@skipif_ibm_power
 @pytest.mark.skip(
     reason="Skipped due to failure in 75% filling-up cluster "
     "which created more PODs and failed for memory issue"
