diff --git a/ocs_ci/deployment/helpers/lso_helpers.py b/ocs_ci/deployment/helpers/lso_helpers.py
index 7e0025ea..cc91d080 100644
--- a/ocs_ci/deployment/helpers/lso_helpers.py
+++ b/ocs_ci/deployment/helpers/lso_helpers.py
@@ -44,9 +44,13 @@ def setup_local_storage(storageclass):
     ocs_version = config.ENV_DATA.get("ocs_version")
     ocp_ga_version = get_ocp_ga_version(ocp_version)
     if not ocp_ga_version:
-        optional_operators_data = templating.load_yaml(
-            constants.LOCAL_STORAGE_OPTIONAL_OPERATORS, multi_document=True
+        optional_operators_data = list(
+            templating.load_yaml(constants.LOCAL_STORAGE_OPTIONAL_OPERATORS, multi_document=True)
         )
+        if config.ENV_DATA["platform"].lower() == constants.IBM_POWER_PLATFORM:
+            for data in optional_operators_data:
+                if data["kind"] == "CatalogSource":
+                    data["spec"]["image"] = config.DEPLOYMENT.get("optional_operators_image")
         logger.info(
             "Creating temp yaml file with optional operators data:\n %s",
             optional_operators_data,
