diff --git a/ocs_ci/deployment/deployment.py b/ocs_ci/deployment/deployment.py
index 9b609f15..b13b6240 100644
--- a/ocs_ci/deployment/deployment.py
+++ b/ocs_ci/deployment/deployment.py
@@ -1132,6 +1132,12 @@ def setup_persistent_monitoring():
     """
     Change monitoring backend to OCS
     """
+    # Validate the storage class exists
+    retry((CommandFailed), tries=16, delay=15)(
+        helpers.default_storage_class
+    )(interface_type=constants.CEPHBLOCKPOOL)
+
+
     sc = helpers.default_storage_class(interface_type=constants.CEPHBLOCKPOOL)

     # Get the list of monitoring pods
