diff --git a/pyproject.toml b/pyproject.toml
index 857122ffd..6c0abda1b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -56,7 +56,7 @@ dependencies = [
     "msrestazure==0.6.3",
     "munch==2.5.0",
     "mysql-connector-python==9.1.0",
-    "numpy==1.23.2",
+    "numpy==2.4.2",
     "oauth2client==4.1.3",
     "ocp-network-split>=0.3.1",
     "openshift==0.13.2",
@@ -65,7 +65,7 @@ dependencies = [
     "oslo-serialization==5.7.0",  # Dependency of python-novaclient and python-cinderclient
     "oslo-utils==9.0.0",  # Dependency of python-novaclient and python-cinderclient
     "ovirt-engine-sdk-python==4.4.11",
-    "pandas==1.5.2",
+    "pandas==3.0.0",
     "paramiko==3.5.0",
     "pexpect>=4.8.0",
     "ping3==4.0.3",
@@ -94,7 +94,7 @@ dependencies = [
     "range-key-dict==1.1.0",
     "reportportal-client==3.2.3",
     "requests==2.32.2",
-    "scipy==1.12.0",
+    "scipy==1.17.0",
     "selenium==4.36.0",
     "semantic-version==2.8.5",
     "tabulate==0.9.0",
