diff --git a/ocs_ci/deployment/deployment.py b/ocs_ci/deployment/deployment.py
index 2c420b756..262bd6ad7 100644
--- a/ocs_ci/deployment/deployment.py
+++ b/ocs_ci/deployment/deployment.py
@@ -2822,6 +2822,11 @@ def setup_persistent_monitoring():
     https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation
     /4.16/html-single/managing_and_allocating_storage_resources/
     """
+    # Validate the storage class exists
+    retry((CommandFailed), tries=16, delay=15)(
+        helpers.default_storage_class
+    )(interface_type=constants.CEPHBLOCKPOOL)
+
     sc = helpers.default_storage_class(interface_type=constants.CEPHBLOCKPOOL)
 
     # Get the list of monitoring pods
