diff --git a/ocs_ci/ocs/node.py b/ocs_ci/ocs/node.py
index f8676cd8..a5fe035b 100644
--- a/ocs_ci/ocs/node.py
+++ b/ocs_ci/ocs/node.py
@@ -213,7 +213,7 @@ def drain_nodes(node_names):
         )
         ocp.exec_oc_cmd(
             f"adm drain {node_names_str} --force=true --ignore-daemonsets "
-            f"{drain_deletion_flag}",
+            f"{drain_deletion_flag} --timeout=1810s",
             timeout=1800,
         )
     except TimeoutExpired:
