diff --git a/ocs_ci/ocs/resources/pod.py b/ocs_ci/ocs/resources/pod.py
index 12d3a936..ffc2abdd 100644
--- a/ocs_ci/ocs/resources/pod.py
+++ b/ocs_ci/ocs/resources/pod.py
@@ -1954,8 +1954,13 @@ def wait_for_storage_pods(timeout=200):
 
     for pod_obj in all_pod_obj:
         state = constants.STATUS_RUNNING
-        if any(i in pod_obj.name for i in ["-1-deploy", "osd-prepare"]):
-            state = constants.STATUS_COMPLETED
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if (ocs_version >= version.VERSION_4_10) and (config.ENV_DATA["platform"].lower() == constants.IBM_POWER_PLATFORM):
+            if any(i in pod_obj.name for i in ["-1-deploy", "osd-prepare"]):
+                state = constants.STATUS_COMPLETED
+        else:
+            if any(i in pod_obj.name for i in ["-1-deploy", "ocs-deviceset"]):
+                state = constants.STATUS_COMPLETED
         helpers.wait_for_resource_state(resource=pod_obj, state=state, timeout=timeout)
 
 

