diff --git a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
index bab85f605..ce90d1560 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
@@ -25,6 +25,9 @@ from ocs_ci.framework import config
 from ocs_ci.helpers.pvc_ops import test_create_delete_pvcs
 from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
 from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
 
 logger = logging.getLogger(__name__)
 
@@ -42,6 +45,7 @@ logger = logging.getLogger(__name__)
 @ignore_leftovers
 @tier2
 @skipif_bm
+@skipif_ibm_power
 @skipif_external_mode
 @skipif_managed_service
 @skipif_hci_provider_and_client
diff --git a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
index 634719222..99003cce7 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
@@ -23,6 +23,9 @@ from ocs_ci.ocs.cluster import (
     is_vsphere_ipi_cluster,
 )
 
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
 logger = logging.getLogger(__name__)
 
 
@@ -38,6 +41,7 @@ logger = logging.getLogger(__name__)
 )
 @ignore_leftovers
 @tier4b
+@skipif_ibm_power
 @skipif_managed_service
 @skipif_hci_provider_and_client
 @skipif_external_mode
diff --git a/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py b/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py
index 62152e78a..b669af5b5 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py
@@ -12,6 +12,7 @@ from ocs_ci.framework.testlib import (
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_external_mode,
     brown_squad,
+    skipif_ibm_power,
 )
 from ocs_ci.framework import config
 from ocs_ci.ocs import constants
@@ -34,6 +35,7 @@ logger = logging.getLogger(__name__)
 @skipif_hci_provider_and_client
 @skipif_external_mode
 @ignore_leftovers
+@skipif_ibm_power
 @tier4c
 class TestAddCapacityWithResourceDelete:
     """
diff --git a/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py b/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py
index 2c0b143a8..d5ea3b788 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py
@@ -8,6 +8,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_managed_service,
     skipif_hci_provider_and_client,
     brown_squad,
+    skipif_ibm_power,
 )
 
 logger = logging.getLogger(__name__)
@@ -20,6 +21,7 @@ logger = logging.getLogger(__name__)
 @skipif_flexy_deployment
 @skipif_ibm_flash
 @ignore_leftovers
+@skipif_ibm_power
 @tier1
 class TestAddNode(ManageTest):
     """
diff --git a/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py b/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
index cd5065f4c..96a10228f 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
@@ -8,6 +8,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_hci_provider_and_client,
     skipif_multus_enabled,
     brown_squad,
+    skipif_ibm_power,
 )
 from ocs_ci.framework.testlib import tier2, ManageTest, ignore_leftovers
 from ocs_ci.ocs.node import get_worker_nodes_not_in_ocs
@@ -25,6 +26,7 @@ logger = logging.getLogger(__name__)
 @tier2
 @pytest.mark.polarion_id("OCS-2490")
 @pytest.mark.bugzilla("1794389")
+@skipif_ibm_power
 @ignore_leftovers
 class TestCheckTolerationForCephCsiDriverDs(ManageTest):
     """
diff --git a/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py b/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
index e3d3615ff..0412a84d1 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
@@ -6,6 +6,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_hci_provider_and_client,
     skipif_bm,
     brown_squad,
+    skipif_ibm_power,
 )
 from ocs_ci.ocs.node import drain_nodes, schedule_nodes, is_node_rack_or_zone_exist
 from ocs_ci.helpers.helpers import get_failure_domin
@@ -40,6 +41,7 @@ logger = logging.getLogger(__name__)
 @skipif_bm
 @skipif_external_mode
 @skipif_managed_service
+@skipif_ibm_power
 @skipif_hci_provider_and_client
 @pytest.mark.polarion_id("OCS-2594")
 class TestAddNodeCrashCollector(ManageTest):
diff --git a/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py b/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py
index a5145d011..1c28ab088 100644
--- a/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py
+++ b/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py
@@ -23,6 +23,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_ms_consumer,
     skipif_hci_client,
     brown_squad,
+    skipif_ibm_power,
 )
 from ocs_ci.helpers.helpers import verify_storagecluster_nodetopology
 from ocs_ci.helpers.sanity_helpers import Sanity
@@ -182,6 +183,7 @@ def delete_and_create_osd_node(osd_node_name):
 @skipif_managed_service
 @skipif_hci_provider_and_client
 @skipif_bmpsi
+@skipif_ibm_power
 @skipif_external_mode
 class TestNodeReplacementWithIO(ManageTest):
     """
@@ -262,6 +264,7 @@ class TestNodeReplacementWithIO(ManageTest):
 @skipif_external_mode
 @skipif_ms_consumer
 @skipif_hci_client
+@skipif_ibm_power
 class TestNodeReplacement(ManageTest):
     """
     Knip-894 Node replacement proactive
@@ -310,6 +313,7 @@ class TestNodeReplacement(ManageTest):
 @skipif_external_mode
 @skipif_managed_service
 @skipif_hci_provider_and_client
+@skipif_ibm_power
 class TestNodeReplacementTwice(ManageTest):
     """
     Node replacement twice:
