diff --git a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
index fb0c8470..af225537 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py
@@ -2,6 +2,10 @@ import logging
 from concurrent.futures import ThreadPoolExecutor
 import pytest

+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
+
 from ocs_ci.ocs.cluster import is_flexible_scaling_enabled
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import pod as pod_helpers
@@ -44,6 +48,7 @@ logger = logging.getLogger(__name__)
 @ignore_leftovers
 @tier2
 @skipif_bm
+@skipif_ibm_power
 @skipif_external_mode
 @skipif_managed_service
 @skipif_hci_provider_and_client
diff --git a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
index 63471922..58e8d88c 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
@@ -1,6 +1,10 @@
 import pytest
 import logging

+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
+
 from ocs_ci.framework.pytest_customization.marks import brown_squad
 from ocs_ci.framework.testlib import (
     ignore_leftovers,
@@ -39,6 +43,7 @@ logger = logging.getLogger(__name__)
 @ignore_leftovers
 @tier4b
 @skipif_managed_service
+@skipif_ibm_power
 @skipif_hci_provider_and_client
 @skipif_external_mode
 class TestAddCapacityNodeRestart(ManageTest):
diff --git a/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py b/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py
index 62152e78..7729cb8e 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_delete_pod.py
@@ -11,6 +11,7 @@ from ocs_ci.framework.testlib import (
 )
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_external_mode,
+    skipif_ibm_power,
     brown_squad,
 )
 from ocs_ci.framework import config
@@ -33,6 +34,7 @@ logger = logging.getLogger(__name__)
 @skipif_managed_service
 @skipif_hci_provider_and_client
 @skipif_external_mode
+@skipif_ibm_power
 @ignore_leftovers
 @tier4c
 class TestAddCapacityWithResourceDelete:
diff --git a/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py b/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py
index 2c0b143a..e59f101a 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_node_expansion.py
@@ -7,6 +7,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_ibm_flash,
     skipif_managed_service,
     skipif_hci_provider_and_client,
+    skipif_ibm_power,
     brown_squad,
 )

@@ -19,6 +20,7 @@ logger = logging.getLogger(__name__)
 @skipif_hci_provider_and_client
 @skipif_flexy_deployment
 @skipif_ibm_flash
+@skipif_ibm_power
 @ignore_leftovers
 @tier1
 class TestAddNode(ManageTest):
diff --git a/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py b/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
index cd5065f4..b073268c 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py
@@ -5,6 +5,7 @@ from ocs_ci.ocs import constants
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_flexy_deployment,
     skipif_managed_service,
+    skipif_ibm_power,
     skipif_hci_provider_and_client,
     skipif_multus_enabled,
     brown_squad,
@@ -21,6 +22,7 @@ logger = logging.getLogger(__name__)
 @skipif_flexy_deployment
 @skipif_managed_service
 @skipif_hci_provider_and_client
+@skipif_ibm_power
 @skipif_multus_enabled
 @tier2
 @pytest.mark.polarion_id("OCS-2490")
diff --git a/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py b/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
index 4dcadbea..50d9c978 100644
--- a/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
+++ b/tests/functional/z_cluster/cluster_expansion/test_crashcollector_pod_existence_on_ceph_pods_running_nodes.py
@@ -5,6 +5,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_managed_service,
     skipif_hci_provider_and_client,
     skipif_bm,
+    skipif_ibm_power,
     brown_squad,
     skipif_compact_mode,
 )
@@ -41,6 +42,7 @@ logger = logging.getLogger(__name__)
 @skipif_bm
 @skipif_external_mode
 @skipif_managed_service
+@skipif_ibm_power
 @skipif_hci_provider_and_client
 @skipif_compact_mode
 @pytest.mark.polarion_id("OCS-2594")

diff --git a/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py b/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py
index 8812511dc..db6616f31 100644
--- a/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py
+++ b/tests/functional/z_cluster/nodes/test_node_replacement_proactive.py
@@ -21,6 +21,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     bugzilla,
     skipif_external_mode,
     skipif_ms_consumer,
+    skipif_ibm_power,
     skipif_hci_client,
     brown_squad,
 )
@@ -184,6 +185,7 @@ def delete_and_create_osd_node(osd_node_name):
 @skipif_managed_service
 @skipif_hci_provider_and_client
 @skipif_bmpsi
+@skipif_ibm_power
 @skipif_external_mode
 class TestNodeReplacementWithIO(ManageTest):
     """
@@ -263,6 +265,7 @@ class TestNodeReplacementWithIO(ManageTest):
 @skipif_bmpsi
 @skipif_external_mode
 @skipif_ms_consumer
+@skipif_ibm_power
 @skipif_hci_client
 class TestNodeReplacement(ManageTest):
     """
@@ -311,6 +314,7 @@ class TestNodeReplacement(ManageTest):
 @pytest.mark.polarion_id("OCS-2535")
 @skipif_external_mode
 @skipif_managed_service
+@skipif_ibm_power
 @skipif_hci_provider_and_client
 class TestNodeReplacementTwice(ManageTest):
     """
