diff --git a/tests/functional/z_cluster/test_ceph_default_values_check.py b/tests/functional/z_cluster/test_ceph_default_values_check.py
index 48f5a5785..da93e087b 100644
--- a/tests/functional/z_cluster/test_ceph_default_values_check.py
+++ b/tests/functional/z_cluster/test_ceph_default_values_check.py
@@ -4,6 +4,7 @@ import pytest

 from ocs_ci.framework.pytest_customization.marks import (
     skipif_ocs_version,
+    skipif_ibm_power,
     brown_squad,
 )
 from ocs_ci.framework.testlib import (
@@ -208,6 +209,7 @@ class TestCephDefaultValuesCheck(ManageTest):
         config.DEPLOYMENT.get("ceph_debug"),
         reason="Ceph was configured with customized values by ocs-ci so there is point in validating its config values",
     )
+    @skipif_ibm_power
     def test_validate_ceph_config_values_in_rook_config_override(self):
         """
         Test case for comparing the cluster's config values of
diff --git a/tests/functional/z_cluster/test_must_gather.py b/tests/functional/z_cluster/test_must_gather.py
index 8ea47fa5c..506ca4836 100644
--- a/tests/functional/z_cluster/test_must_gather.py
+++ b/tests/functional/z_cluster/test_must_gather.py
@@ -1,7 +1,7 @@
 import logging
 import pytest

-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ManageTest,
     tier1,
@@ -29,6 +29,7 @@ def mustgather(request):


 @brown_squad
+@skipif_ibm_power
 class TestMustGather(ManageTest):
     @tier1
     @pytest.mark.parametrize(
diff --git a/tests/functional/object/mcg/test_s3_with_java_sdk.py b/tests/functional/object/mcg/test_s3_with_java_sdk.py
index b32166151..86fbfd16b 100644
--- a/tests/functional/object/mcg/test_s3_with_java_sdk.py
+++ b/tests/functional/object/mcg/test_s3_with_java_sdk.py
@@ -4,6 +4,7 @@ import pytest
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_ocs_version,
     skipif_disconnected_cluster,
+    skipif_ibm_power,
     skipif_proxy_cluster,
     tier1,
     red_squad,
@@ -20,6 +21,7 @@ logger = logging.getLogger(__name__)
 @runs_on_provider
 @skipif_ocs_version("<4.9")
 @skipif_disconnected_cluster
+@skipif_ibm_power
 @skipif_proxy_cluster
 class TestS3WithJavaSDK:
     @pytest.mark.parametrize(
diff --git a/tests/functional/object/mcg/test_noobaa_secret.py b/tests/functional/object/mcg/test_noobaa_secret.py
index 21375796e..0a96ed066 100644
--- a/tests/functional/object/mcg/test_noobaa_secret.py
+++ b/tests/functional/object/mcg/test_noobaa_secret.py
@@ -20,6 +20,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     polarion_id,
     skipif_ocs_version,
     skipif_disconnected_cluster,
+    skipif_ibm_power,
     red_squad,
     runs_on_provider,
     mcg,
@@ -183,6 +184,7 @@ class TestNoobaaSecrets:
             "Secret remains even after the linked backingstores are deleted, as expected!"
         )

+    @skipif_ibm_power
     @polarion_id("OCS-4468")
     def test_noobaa_secret_deletion_method2(self, teardown_factory, mcg_obj, cleanup):
         """
diff --git a/tests/functional/z_cluster/test_hugepages.py b/tests/functional/z_cluster/test_hugepages.py
index 69d31845d..8e5d88f01 100644
--- a/tests/functional/z_cluster/test_hugepages.py
+++ b/tests/functional/z_cluster/test_hugepages.py
@@ -15,7 +15,7 @@ from ocs_ci.ocs.node import (
 from ocs_ci.ocs.resources.pod import (
     wait_for_pods_to_be_running,
 )
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     skipif_external_mode,
     skipif_ocs_version,
@@ -68,6 +68,7 @@ class TestHugePages(E2ETest):
         request.addfinalizer(finalizer)

     @skipif_external_mode
+    @skipif_ibm_power
     @pytest.mark.order(-3)
     def test_hugepages_post_odf_deployment(
         self,
diff --git a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
-index 592f057e..7753a1f1 100644
+index 8ac47782..56601bbb 100644
 --- a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
 +++ b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
-@@ -10,7 +10,7 @@ from ocs_ci.helpers.helpers import (
-     get_last_log_time_date,
+@@ -11,7 +11,7 @@ from ocs_ci.helpers.helpers import (
      check_osd_log_exist_on_rook_ceph_operator_pod,
  )
+ from ocs_ci.helpers.odf_cli import ODFCLIRetriever, ODFCliRunner
 -from ocs_ci.framework.pytest_customization.marks import brown_squad
-+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
++from ocs_ci.framework.pytest_customization.marks import brown_squad,skipif_ibm_power
  from ocs_ci.framework.testlib import (
      ManageTest,
      tier2,
-@@ -29,6 +29,7 @@ log = logging.getLogger(__name__)
+@@ -30,6 +30,7 @@ log = logging.getLogger(__name__)
 @bugzilla("1962821")
 @skipif_ocs_version("<4.8")
 @skipif_external_mode
+@skipif_ibm_power
 @pytest.mark.polarion_id("OCS-2581")
 class TestRookCephOperatorLogType(ManageTest):
     """
git diff tests/functional/monitoring/prometheus/alerts/test_noobaa.py
diff --git a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
index 3df6bd66c..4333c444b 100644
--- a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
+++ b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
@@ -1,7 +1,7 @@
 import logging

 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import blue_squad
+from ocs_ci.framework.pytest_customization.marks import blue_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     polarion_id,
     post_upgrade,
@@ -29,6 +29,7 @@ log = logging.getLogger(__name__)
 @runs_on_provider
 @skipif_disconnected_cluster
 @skipif_aws_creds_are_missing
+@skipif_ibm_power
 def test_noobaa_bucket_quota(measure_noobaa_exceed_bucket_quota, threading_lock):
     """
     Test that there are appropriate alerts when NooBaa Bucket Quota is reached.
diff --git a/tests/functional/object/mcg/test_mcg_resources_disruptions.py b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
index 2e0df6e51..502025807 100644
--- a/tests/functional/object/mcg/test_mcg_resources_disruptions.py
+++ b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
@@ -17,6 +17,7 @@ from ocs_ci.framework.testlib import (
     runs_on_provider,
     mcg,
 )
+from ocs_ci.framework.pytest_customization.marks import skipif_ibm_power
 from ocs_ci.helpers import helpers
 from ocs_ci.helpers.helpers import wait_for_resource_state
 from ocs_ci.ocs import cluster, constants, ocp
@@ -123,7 +124,13 @@ class TestMCGResourcesDisruptions(MCGTest):
         argnames=["pod_to_drain"],
         argvalues=[
             pytest.param(*["noobaa_core"], marks=pytest.mark.polarion_id("OCS-2286")),
-            pytest.param(*["noobaa_db"], marks=pytest.mark.polarion_id("OCS-2287")),
+            pytest.param(
+                *["noobaa_db"],
+                marks=[
+                    pytest.mark.polarion_id("OCS-2287"),
+                    skipif_ibm_power,
+                ]
+            ),
             pytest.param(
                 *["noobaa_endpoint"], marks=pytest.mark.polarion_id("OCS-2288")
             ),
@@ -225,6 +232,7 @@ class TestMCGResourcesDisruptions(MCGTest):
     @tier3
     @pytest.mark.polarion_id("OCS-2513")
     @skipif_managed_service
+    @skipif_ibm_power
     @skipif_ocs_version("<4.7")
     def deprecated_test_db_scc(self, teardown):
         """
diff --git a/tests/functional/object/mcg/test_host_node_failure.py b/tests/functional/object/mcg/test_host_node_failure.py
index 268b776e0..97dc4996d 100644
--- a/tests/functional/object/mcg/test_host_node_failure.py
+++ b/tests/functional/object/mcg/test_host_node_failure.py
@@ -3,7 +3,7 @@ import pytest
 from concurrent.futures import ThreadPoolExecutor

 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import red_squad, mcg
+from ocs_ci.framework.pytest_customization.marks import red_squad, mcg, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ignore_leftovers,
     ManageTest,
@@ -80,7 +80,10 @@ class TestNoobaaSTSHostNodeFailure(ManageTest):
             ),
             pytest.param(
                 *[constants.NOOBAA_DB_STATEFULSET, False],
-                marks=pytest.mark.polarion_id("OCS-2668"),
+                marks=[
+                    pytest.mark.polarion_id("OCS-2668"),
+                    skipif_ibm_power,
+                ]
             ),
             pytest.param(
                 *[constants.NOOBAA_CORE_STATEFULSET, True],
@@ -88,7 +91,10 @@ class TestNoobaaSTSHostNodeFailure(ManageTest):
             ),
             pytest.param(
                 *[constants.NOOBAA_DB_STATEFULSET, True],
-                marks=pytest.mark.polarion_id("OCS-2670"),
+                marks=[
+                    pytest.mark.polarion_id("OCS-2670"),
+                    skipif_ibm_power,
+                ]
             ),
         ],
     )
diff --git a/tests/functional/z_cluster/nodes/test_nodes_restart.py b/tests/functional/z_cluster/nodes/test_nodes_restart.py
index 988a36439..22b3aa315 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_restart.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_restart.py
@@ -1,7 +1,7 @@
 import logging
 import pytest

-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     tier4a,
     tier4b,
@@ -145,6 +145,7 @@ class TestNodesRestart(ManageTest):
     )
     @skipif_ibm_cloud
     @skipif_vsphere_ipi
+    @skipif_ibm_power
     def test_pv_provisioning_under_degraded_state_stop_provisioner_pod_node(
         self,
         nodes,

diff --git a/tests/functional/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive.py b/tests/functional/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive.py
index ff3b7e1da..ebca9b15f 100644
--- a/tests/functional/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive.py
+++ b/tests/functional/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive.py
@@ -53,6 +53,7 @@ deployment_type = config.ENV_DATA["deployment_type"].lower()
 @brown_squad
 @ignore_leftovers
 @tier4b
+@skipif_ibm_power
 class TestAutomatedRecoveryFromFailedNodes(ManageTest):
     """
     Knip-678 Automated recovery from failed nodes - Reactive
diff --git a/tests/functional/object/rgw/test_keda_ha.py b/tests/functional/object/rgw/test_keda_ha.py
index 8148558b0..a80833ec1 100644
--- a/tests/functional/object/rgw/test_keda_ha.py
+++ b/tests/functional/object/rgw/test_keda_ha.py
@@ -10,6 +10,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_disconnected_cluster,
     skipif_external_mode,
     tier1,
+    skipif_ibm_power,
 )
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.exceptions import TimeoutExpiredError
@@ -32,6 +33,7 @@ RGW_SCALE_TARGET_REF = {
 @rgw
 @red_squad
 @skipif_disconnected_cluster
+@skipif_ibm_power
 @skipif_external_mode  # Setting up KEDA on the external cluster is out of scope
 class TestKedaHA:
     """
diff --git a/tests/functional/z_cluster/test_ceph_mon_healthcheck.py b/tests/functional/z_cluster/test_ceph_mon_healthcheck.py
index b8cea5057..db96517a1 100644
--- a/tests/functional/z_cluster/test_ceph_mon_healthcheck.py
+++ b/tests/functional/z_cluster/test_ceph_mon_healthcheck.py
@@ -27,12 +27,13 @@ from ocs_ci.ocs.node import (
     schedule_nodes,
     get_node_objs,
     recover_node_to_ready_state,
-)
+) 
+from ocs_ci.framework.pytest_customization.marks import skipif_ibm_power
 from ocs_ci.ocs import constants
 
 log = logging.getLogger(__name__)
 
-
+@skipif_ibm_power
 @brown_squad
 @ignore_leftovers
 class TestCephMonHealthCheck(ManageTest):
diff --git a/tests/functional/object/mcg/test_s3_vector.py b/tests/functional/object/mcg/test_s3_vector.py
index 811ca423b..8ae307e39 100644
--- a/tests/functional/object/mcg/test_s3_vector.py
+++ b/tests/functional/object/mcg/test_s3_vector.py
@@ -11,6 +11,7 @@ from botocore.exceptions import ClientError, ParamValidationError

 from ocs_ci.framework.pytest_customization.marks import (
     tier2,
+    skipif_ibm_power,
     red_squad,
     post_upgrade,
     mcg,
@@ -27,6 +28,7 @@ logger = logging.getLogger(__name__)


 @mcg
+@skipif_ibm_power
 @red_squad
 class TestS3Vector(MCGTest):
     """
diff --git a/tests/functional/pv/pv_services/test_automate_networkfence_workflow_with_cephcsi.py b/tests/functional/pv/pv_services/test_automate_networkfence_workflow_with_cephcsi.py
index 787bfec7b..036ec7a25 100644
--- a/tests/functional/pv/pv_services/test_automate_networkfence_workflow_with_cephcsi.py
+++ b/tests/functional/pv/pv_services/test_automate_networkfence_workflow_with_cephcsi.py
@@ -14,6 +14,7 @@ import pytest
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_hci_provider_or_client,
     skipif_managed_service,
+    skipif_ibm_power,
 )
 from ocs_ci.framework.testlib import ManageTest, tier4b, green_squad
 from ocs_ci.ocs import constants
@@ -67,6 +68,7 @@ def pods_for_test_workload_pvcs(pod_obj_list, wait=True):
 @tier4b
 @skipif_managed_service
 @skipif_hci_provider_or_client
+@skipif_ibm_power
 class TestAutomateNetworkfenceWorkflowWithCephCSI(ManageTest):
     """
     RHSTOR-6942 - Automate the NetworkFence workflow with cephcsi.
