diff --git a/tests/functional/z_cluster/test_ceph_default_values_check.py b/tests/functional/z_cluster/test_ceph_default_values_check.py
index 7169a533b..76e094287 100644
--- a/tests/functional/z_cluster/test_ceph_default_values_check.py
+++ b/tests/functional/z_cluster/test_ceph_default_values_check.py
@@ -4,6 +4,7 @@ import pytest

 from ocs_ci.framework.pytest_customization.marks import (
     skipif_ocs_version,
+    skipif_ibm_power,
     brown_squad,
 )
 from ocs_ci.framework.testlib import (
@@ -101,6 +102,7 @@ class TestCephDefaultValuesCheck(ManageTest):
         config.DEPLOYMENT.get("ceph_debug"),
         reason="Ceph was configured with customized values by ocs-ci so there is point in validating its config values",
     )
+    @skipif_ibm_power
     def test_validate_ceph_config_values_in_rook_config_override(self):
         """
         Test case for comparing the cluster's config values of
diff --git a/tests/functional/z_cluster/test_must_gather.py b/tests/functional/z_cluster/test_must_gather.py
index 2685a8b2..f76620d0 100644
--- a/tests/functional/z_cluster/test_must_gather.py
+++ b/tests/functional/z_cluster/test_must_gather.py
@@ -2,7 +2,7 @@ import logging
 import pytest

 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ManageTest,
     tier1,
@@ -53,7 +53,13 @@ class TestMustGather(ManageTest):
                     skipif_hci_client,
                 ]
             ),
-            pytest.param(*["OTHERS"], marks=pytest.mark.polarion_id("OCS-1583")),
+            pytest.param(
+                *["OTHERS"],
+                marks=[
+                    pytest.mark.polarion_id("OCS-1583"),
+                    skipif_ibm_power,
+                ]
+            ),
         ],
     )
     @pytest.mark.skipif(
diff --git a/tests/functional/object/mcg/test_s3_with_java_sdk.py b/tests/functional/object/mcg/test_s3_with_java_sdk.py
index b32166151..86fbfd16b 100644
--- a/tests/functional/object/mcg/test_s3_with_java_sdk.py
+++ b/tests/functional/object/mcg/test_s3_with_java_sdk.py
@@ -4,6 +4,7 @@ import pytest
 from ocs_ci.framework.pytest_customization.marks import (
     skipif_ocs_version,
     skipif_disconnected_cluster,
+    skipif_ibm_power,
     skipif_proxy_cluster,
     tier1,
     red_squad,
@@ -20,6 +21,7 @@ logger = logging.getLogger(__name__)
 @runs_on_provider
 @skipif_ocs_version("<4.9")
 @skipif_disconnected_cluster
+@skipif_ibm_power
 @skipif_proxy_cluster
 class TestS3WithJavaSDK:
     @pytest.mark.parametrize(
diff --git a/tests/functional/object/mcg/test_noobaa_secret.py b/tests/functional/object/mcg/test_noobaa_secret.py
index 21375796e..0a96ed066 100644
--- a/tests/functional/object/mcg/test_noobaa_secret.py
+++ b/tests/functional/object/mcg/test_noobaa_secret.py
@@ -20,6 +20,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     polarion_id,
     skipif_ocs_version,
     skipif_disconnected_cluster,
+    skipif_ibm_power,
     red_squad,
     runs_on_provider,
     mcg,
@@ -183,6 +184,7 @@ class TestNoobaaSecrets:
             "Secret remains even after the linked backingstores are deleted, as expected!"
         )

+    @skipif_ibm_power
     @polarion_id("OCS-4468")
     def test_noobaa_secret_deletion_method2(self, teardown_factory, mcg_obj, cleanup):
         """
diff --git a/tests/functional/z_cluster/test_hugepages.py b/tests/functional/z_cluster/test_hugepages.py
index 69d31845d..8e5d88f01 100644
--- a/tests/functional/z_cluster/test_hugepages.py
+++ b/tests/functional/z_cluster/test_hugepages.py
@@ -15,7 +15,7 @@ from ocs_ci.ocs.node import (
 from ocs_ci.ocs.resources.pod import (
     wait_for_pods_to_be_running,
 )
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     skipif_external_mode,
     skipif_ocs_version,
@@ -68,6 +68,7 @@ class TestHugePages(E2ETest):
         request.addfinalizer(finalizer)

     @skipif_external_mode
+    @skipif_ibm_power
     @pytest.mark.order(-3)
     def test_hugepages_post_odf_deployment(
         self,
diff --git a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
-index 592f057e..7753a1f1 100644
+index 8ac47782..56601bbb 100644
 --- a/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
 +++ b/tests/functional/z_cluster/test_rook_ceph_operator_log_type.py
-@@ -10,7 +10,7 @@ from ocs_ci.helpers.helpers import (
-     get_last_log_time_date,
+@@ -11,7 +11,7 @@ from ocs_ci.helpers.helpers import (
      check_osd_log_exist_on_rook_ceph_operator_pod,
  )
+ from ocs_ci.helpers.odf_cli import ODFCLIRetriever, ODFCliRunner
 -from ocs_ci.framework.pytest_customization.marks import brown_squad
-+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
++from ocs_ci.framework.pytest_customization.marks import brown_squad,skipif_ibm_power
  from ocs_ci.framework.testlib import (
      ManageTest,
      tier2,
-@@ -29,6 +29,7 @@ log = logging.getLogger(__name__)
+@@ -30,6 +30,7 @@ log = logging.getLogger(__name__)
 @bugzilla("1962821")
 @skipif_ocs_version("<4.8")
 @skipif_external_mode
+@skipif_ibm_power
 @pytest.mark.polarion_id("OCS-2581")
 class TestRookCephOperatorLogType(ManageTest):
     """
diff --git a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
index a2f706fb..a4e42259 100644
--- a/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
+++ b/tests/functional/monitoring/prometheus/alerts/test_noobaa.py
@@ -1,6 +1,6 @@
 import logging

-from ocs_ci.framework.pytest_customization.marks import blue_squad
+from ocs_ci.framework.pytest_customization.marks import blue_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     polarion_id,
     skipif_aws_creds_are_missing,
@@ -24,6 +24,7 @@ log = logging.getLogger(__name__)
 @skipif_managed_service
 @skipif_disconnected_cluster
 @skipif_aws_creds_are_missing
+@skipif_ibm_power
 def test_noobaa_bucket_quota(measure_noobaa_exceed_bucket_quota, threading_lock):
     """
     Test that there are appropriate alerts when NooBaa Bucket Quota is reached.
diff --git a/tests/functional/object/mcg/test_mcg_resources_disruptions.py b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
index af9c252a9..a384e16b2 100644
--- a/tests/functional/object/mcg/test_mcg_resources_disruptions.py
+++ b/tests/functional/object/mcg/test_mcg_resources_disruptions.py
@@ -17,6 +17,7 @@ from ocs_ci.framework.testlib import (
     runs_on_provider,
     mcg,
 )
+from ocs_ci.framework.pytest_customization.marks import skipif_ibm_power
 from ocs_ci.helpers import helpers
 from ocs_ci.helpers.helpers import wait_for_resource_state
 from ocs_ci.ocs import cluster, constants, ocp
@@ -129,7 +130,13 @@ class TestMCGResourcesDisruptions(MCGTest):
         argnames=["pod_to_drain"],
         argvalues=[
             pytest.param(*["noobaa_core"], marks=pytest.mark.polarion_id("OCS-2286")),
-            pytest.param(*["noobaa_db"], marks=pytest.mark.polarion_id("OCS-2287")),
+            pytest.param(
+                *["noobaa_db"],
+                marks=[
+                    pytest.mark.polarion_id("OCS-2287"),
+                    skipif_ibm_power,
+                ]
+            ),
             pytest.param(
                 *["noobaa_endpoint"], marks=pytest.mark.polarion_id("OCS-2288")
             ),
@@ -231,6 +238,7 @@ class TestMCGResourcesDisruptions(MCGTest):
     @tier3
     @pytest.mark.polarion_id("OCS-2513")
     @skipif_managed_service
+    @skipif_ibm_power
     @skipif_ocs_version("<4.7")
     def test_db_scc(self, teardown):
         """
diff --git a/tests/functional/object/mcg/test_host_node_failure.py b/tests/functional/object/mcg/test_host_node_failure.py
index 10d1bfb75..6539282ed 100644
--- a/tests/functional/object/mcg/test_host_node_failure.py
+++ b/tests/functional/object/mcg/test_host_node_failure.py
@@ -3,7 +3,7 @@ import pytest
 from concurrent.futures import ThreadPoolExecutor

 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import red_squad, mcg
+from ocs_ci.framework.pytest_customization.marks import red_squad, mcg, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     ignore_leftovers,
     ManageTest,
@@ -69,7 +69,10 @@ class TestNoobaaSTSHostNodeFailure(ManageTest):
             ),
             pytest.param(
                 *[constants.NOOBAA_DB_STATEFULSET, False],
-                marks=pytest.mark.polarion_id("OCS-2668"),
+                marks=[
+                    pytest.mark.polarion_id("OCS-2668"),
+                    skipif_ibm_power,
+                ]
             ),
             pytest.param(
                 *[constants.NOOBAA_CORE_STATEFULSET, True],
@@ -77,7 +80,10 @@ class TestNoobaaSTSHostNodeFailure(ManageTest):
             ),
             pytest.param(
                 *[constants.NOOBAA_DB_STATEFULSET, True],
-                marks=pytest.mark.polarion_id("OCS-2670"),
+                marks=[
+                    pytest.mark.polarion_id("OCS-2670"),
+                    skipif_ibm_power,
+                ]
             ),
         ],
     )
diff --git a/tests/functional/z_cluster/nodes/test_nodes_restart.py b/tests/functional/z_cluster/nodes/test_nodes_restart.py
index ec2a881ef..0d3e2feb5 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_restart.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_restart.py
@@ -1,7 +1,7 @@
 import logging
 import pytest

-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     tier4a,
     tier4b,
@@ -139,6 +139,7 @@ class TestNodesRestart(ManageTest):
     )
     @skipif_ibm_cloud
     @skipif_vsphere_ipi
+    @skipif_ibm_power
     def test_pv_provisioning_under_degraded_state_stop_provisioner_pod_node(
         self,
         nodes,
