diff --git a/ocs_ci/utility/localstorage.py b/ocs_ci/utility/localstorage.py
index ea50b40a..94d331fd 100644
--- a/ocs_ci/utility/localstorage.py
+++ b/ocs_ci/utility/localstorage.py
@@ -226,6 +226,6 @@ def get_lso_channel():
     else:
         if len(sorted_versions) >= 1:
             # Use latest channel
-            return sorted_versions[-1]
+            return "stable"
         else:
             return channels[-1]["name"]
