diff --git a/ocs_ci/ocs/node.py b/ocs_ci/ocs/node.py
index eb03446f..fdf6eb0c 100644
--- a/ocs_ci/ocs/node.py
+++ b/ocs_ci/ocs/node.py
@@ -214,7 +214,7 @@ def drain_nodes(node_names):
         )
         ocp.exec_oc_cmd(
             f"adm drain {node_names_str} --force=true --ignore-daemonsets "
-            f"{drain_deletion_flag}",
+            f"{drain_deletion_flag} --timeout=1810s",
             timeout=1800,
         )
     except TimeoutExpired:
