diff --git a/tests/manage/pv_services/space_reclaim/test_rbd_space_reclaim.py b/tests/manage/pv_services/space_reclaim/test_rbd_space_reclaim.py
index 2de0a838..c4b56120 100644
--- a/tests/manage/pv_services/space_reclaim/test_rbd_space_reclaim.py
+++ b/tests/manage/pv_services/space_reclaim/test_rbd_space_reclaim.py
@@ -1,5 +1,6 @@
 import logging
 import pytest
+from time import sleep

 from ocs_ci.ocs import constants
 from ocs_ci.framework.testlib import (
@@ -105,6 +106,7 @@ class TestRbdSpaceReclaim(ManageTest):
                 raise
             log.info(f"Verified: File {file_path} deleted.")

+        sleep(30)
         # Create ReclaimSpaceJob
         reclaim_space_job = pvc_obj.create_reclaim_space_job()

