diff --git a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
index e184469b8..c7dd75cc0 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
@@ -56,6 +56,9 @@ from ocs_ci.helpers.helpers import (
 )
 from ocs_ci.helpers import helpers
 
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ibm_power,
+)
 
 log = logging.getLogger(__name__)
 
@@ -134,6 +137,7 @@ class TestNodesMaintenance(ManageTest):
     @tier1
     @skipif_managed_service
     @skipif_hci_provider_and_client
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["node_type"],
         argvalues=[
@@ -213,6 +217,7 @@ class TestNodesMaintenance(ManageTest):
     @skipif_bm
     @skipif_managed_service
     @skipif_hci_provider_and_client
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["node_type"],
         argvalues=[
@@ -493,6 +498,7 @@ class TestNodesMaintenance(ManageTest):
     @skipif_managed_service
     @skipif_hci_provider_and_client
     @skipif_more_than_three_workers
+    @skipif_ibm_power
     @pytest.mark.polarion_id("OCS-2524")
     @tier4a
     def test_pdb_check_simultaneous_node_drains(
