diff --git a/ocs_ci/helpers/helpers.py b/ocs_ci/helpers/helpers.py
index c03f840c4..c150aafc6 100644
--- a/ocs_ci/helpers/helpers.py
+++ b/ocs_ci/helpers/helpers.py
@@ -109,7 +109,7 @@ def create_resource(do_reload=True, **kwargs):
     return ocs_obj
 
 
-def wait_for_resource_state(resource, state, timeout=60):
+def wait_for_resource_state(resource, state, timeout=120):
     """
     Wait for a resource to get to a given status
 
