diff --git a/ocs_ci/ocs/cluster.py b/ocs_ci/ocs/cluster.py
index 60a25e4aa..f22e7ec30 100644
--- a/ocs_ci/ocs/cluster.py
+++ b/ocs_ci/ocs/cluster.py
@@ -3621,7 +3621,7 @@ def set_osd_op_complaint_time(osd_op_complaint_time_val: float) -> dict:
     """
     ct_pod = pod.get_ceph_tools_pod()
     return ct_pod.exec_ceph_cmd(
-        f"ceph config set osd osd_op_complaint_time {osd_op_complaint_time_val}"
+        f"ceph tell osd.* config set osd_op_complaint_time {osd_op_complaint_time_val}"
     )
 
 
