diff --git a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
index b2566c107..a7b917557 100644
--- a/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
+++ b/tests/functional/z_cluster/nodes/test_nodes_maintenance.py
@@ -32,6 +32,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     skipif_hci_provider,
     skipif_rosa_hcp,
     skipif_compact_mode,
+    skipif_ibm_power,
 )
 from ocs_ci.framework.testlib import (
     tier1,
@@ -135,6 +136,7 @@ class TestNodesMaintenance(ManageTest):
     @tier1
     @skipif_managed_service
     @skipif_hci_provider
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["node_type"],
         argvalues=[
@@ -214,6 +216,7 @@ class TestNodesMaintenance(ManageTest):
     @skipif_bm
     @skipif_managed_service
     @skipif_hci_provider_and_client
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["node_type"],
         argvalues=[
@@ -373,6 +376,7 @@ class TestNodesMaintenance(ManageTest):
     @tier4a
     @aws_based_platform_required
     @ipi_deployment_required
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["interface"],
         argvalues=[
@@ -496,6 +500,7 @@ class TestNodesMaintenance(ManageTest):
     @skipif_more_than_three_workers
     @pytest.mark.polarion_id("OCS-2524")
     @tier4a
+    @skipif_ibm_power
     def test_pdb_check_simultaneous_node_drains(
         self,
         pvc_factory,
