diff --git a/ocs_ci/ocs/ripsaw.py b/ocs_ci/ocs/ripsaw.py
index 032e541f..ee3e1ffb 100644
--- a/ocs_ci/ocs/ripsaw.py
+++ b/ocs_ci/ocs/ripsaw.py
@@ -74,6 +74,15 @@ class RipSaw(object):
             run(git_clone_cmd, shell=True, cwd=self.dir, check=True)
             self.crd = "resources/crds/"
             self.operator = "resources/operator.yaml"
+
+            self.projdir = self.dir + "/benchmark-operator/resources"
+            log.info(f"Patching ripsaw 1 for ppc64le in {self.projdir}")
+            run("sed -i 's|quay.io/benchmark-operator/benchmark-operator:master|quay.io/multi-arch/benchmark-operator:latest|g' operator.yaml" , shell=True, check=True, cwd=self.projdir)
+            log.info(f"Patching ripsaw 2 for ppc64le in {self.projdir}")
+            run("sed -i 's|k8s.gcr.io/redis:v1|quay.io/multiarch-k8s-e2e/redis:1.1|g' operator.yaml", shell=True, check=True, cwd=self.projdir)
+            log.info(f"Patching ripsaw 3 for ppc64le in {self.projdir}")
+            run("sed -i 's|quay.io/cloud-bulldozer/backpack:latest|quay.io/multi-arch/cloud-bulldozer:backpack|g' crds/ripsaw_v1alpha1_ripsaw_crd.yaml", shell=True, check=True, cwd=self.projdir)
+
         except (CommandFailed, CalledProcessError) as cf:
             log.error("Error during cloning of ripsaw repository")
             raise cf
