diff --git a/ocs_ci/ocs/benchmark_operator.py b/ocs_ci/ocs/benchmark_operator.py
index 2a1c324e..ec60c78d 100644
--- a/ocs_ci/ocs/benchmark_operator.py
+++ b/ocs_ci/ocs/benchmark_operator.py
@@ -119,6 +119,19 @@ class BenchmarkOperator(object):
             log.info(f"Cloning {BMO_NAME} in {self.dir}")
             git_clone_cmd = f"git clone -b {self.branch} {self.repo} --depth 1"
             run(git_clone_cmd, shell=True, cwd=self.dir, check=True)
+
+            self.projdir = self.dir + "/benchmark-operator/config"
+            log.info(f"Patching kube-burner for ppc64le in {self.projdir}")
+            run("sed -i 's|quay.io/cloud-bulldozer/kube-burner:latest|quay.io/multi-arch/cloud-bulldozer:kube-burner|g' crd/bases/ripsaw.cloudbulldozer.io_benchmarks.yaml" , shell=True, check=True, cwd=self.projdir)
+            log.info(f"Patching backpack for ppc64le in {self.projdir}")
+            run("sed -i 's|quay.io/cloud-bulldozer/backpack:latest|quay.io/multi-arch/cloud-bulldozer:backpack|g' crd/bases/ripsaw.cloudbulldozer.io_benchmarks.yaml", shell=True, check=True, cwd=self.projdir)
+            log.info(f"Patching redis for ppc64le in {self.projdir}")
+            run("sed -i 's|bitnami/redis:latest|quay.io/multiarch-k8s-e2e/redis:1.1|g' manager/manager.yaml", shell=True, check=True, cwd=self.projdir)
+
+        #    self.projdir1 = self.dir + "/benchmark-operator/roles"
+        #    log.info(f"Patching pgbench for ppc64le in {self.projdir1}")
+        #    run("sed -i 's|quay.io/cloud-bulldozer/pgbench:latest|quay.io/multi-arch/cloud-bulldozer:pgbench|g' pgbench/defaults/main.yml", shell=True, check=True, cwd=self.projdir1)
+
         except (CommandFailed, CalledProcessError) as cf:
             log.error(f"Error during cloning of {BMO_NAME} repository")
             raise cf
@@ -150,7 +163,7 @@ class BenchmarkOperator(object):
         log.info("Deploy the benchmark-operator project")
         try:
             run(
-                "make deploy IMG=quay.io/ocsci/benchmark-operator:testing",
+                "make deploy IMG=quay.io/aaruniaggarwal/benchmark-operator:latest",
                 shell=True,
                 check=True,
                 cwd=self.dir,
