diff --git a/ocs_ci/helpers/helpers.py b/ocs_ci/helpers/helpers.py
index 9554ba0cd..3bd2b5a48 100644
--- a/ocs_ci/helpers/helpers.py
+++ b/ocs_ci/helpers/helpers.py
@@ -1953,6 +1953,16 @@ def change_default_storageclass(scname):
     """
     default_sc = get_default_storage_class()
     ocp_obj = ocp.OCP(kind="StorageClass")
+
+    # Change the new storageclass to default
+    patch = (
+        ' \'{"metadata": {"annotations":'
+        '{"storageclass.kubernetes.io/is-default-class"'
+        ':"true"}}}\' '
+    )
+    patch_cmd = f"patch storageclass {scname} -p" + patch
+    ocp_obj.exec_oc_cmd(command=patch_cmd)
+
     if default_sc:
         # Change the existing default Storageclass annotation to false
         for sc in default_sc:
@@ -1964,16 +1974,6 @@ def change_default_storageclass(scname):
             patch_cmd = f"patch storageclass {sc} -p" + patch
             ocp_obj.exec_oc_cmd(command=patch_cmd)
 
-    # Change the new storageclass to default
-    patch = (
-        ' \'{"metadata": {"annotations":'
-        '{"storageclass.kubernetes.io/is-default-class"'
-        ':"true"}}}\' '
-    )
-    patch_cmd = f"patch storageclass {scname} -p" + patch
-    ocp_obj.exec_oc_cmd(command=patch_cmd)
-    return True
-
 
 def is_volume_present_in_backend(interface, image_uuid, pool_name=None):
     """
