diff --git a/setup.py b/setup.py
index 8ef9aa261..fb69792ef 100644
--- a/setup.py
+++ b/setup.py
@@ -48,11 +48,11 @@ setup(
         "google-cloud-storage==2.6.0",
         "google-auth==2.14.1",
         "elasticsearch==8.11.1",
-        "numpy==1.23.2",
+        "numpy==1.26.4",
         "pandas==1.5.2",
         "tabulate==0.9.0",
         "python-ipmi==0.4.2",
-        "scipy==1.12.0",
+        "scipy==1.11.0",
         "PrettyTable==0.7.2",
         "azure-common==1.1.28",
         "azure-mgmt-compute==33.0.0",
