diff --git a/tests/cross_functional/scale/test_pv_scale_and_respin_ceph_pods.py b/tests/cross_functional/scale/test_pv_scale_and_respin_ceph_pods.py
index 056b9ed79..69667a2d2 100644
--- a/tests/cross_functional/scale/test_pv_scale_and_respin_ceph_pods.py
+++ b/tests/cross_functional/scale/test_pv_scale_and_respin_ceph_pods.py
@@ -15,7 +15,10 @@ from ocs_ci.helpers import helpers, disruption_helpers
 from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
 from ocs_ci.framework.pytest_customization.marks import orange_squad
 from ocs_ci.framework.testlib import scale, E2ETest, ignore_leftovers
-from ocs_ci.framework.pytest_customization.marks import skipif_external_mode
+from ocs_ci.framework.pytest_customization.marks import (
+        skipif_external_mode,
+        skipif_ibm_power,
+)
 from ocs_ci.ocs.exceptions import (
     PVCNotCreated,
     PodNotCreated,
@@ -231,6 +234,7 @@ class BasePvcCreateRespinCephPods(E2ETest):
 @orange_squad
 @scale
 @ignore_leftovers
+@skipif_ibm_power
 @skipif_external_mode
 @pytest.mark.parametrize(
     argnames="resource_to_delete",
