diff --git a/ocs_ci/ocs/ocp.py b/ocs_ci/ocs/ocp.py
index 5b1079d9..108d06ae 100644
--- a/ocs_ci/ocs/ocp.py
+++ b/ocs_ci/ocs/ocp.py
@@ -656,7 +656,7 @@ class OCP(object):
 
         return False
 
-    def wait_for_delete(self, resource_name="", timeout=60, sleep=3):
+    def wait_for_delete(self, resource_name="", timeout=60, sleep=6):
         """
         Wait for a resource to be deleted
 
diff --git a/ocs_ci/ocs/ripsaw.py b/ocs_ci/ocs/ripsaw.py
index d1bf325b..14dce6ac 100644
--- a/ocs_ci/ocs/ripsaw.py
+++ b/ocs_ci/ocs/ripsaw.py
@@ -41,7 +41,7 @@ class RipSaw(object):
         """
         self.args = kwargs
         self.repo = self.args.get(
-            "repo", "https://github.com/cloud-bulldozer/benchmark-operator"
+            "repo", "https://github.com/ocp-power-automation/benchmark-operator"
         )
         self.branch = self.args.get("branch", "master")
         self.namespace = self.args.get("namespace", RIPSAW_NAMESPACE)
diff --git a/ocs_ci/templates/workloads/fio/benchmark_fio.yaml b/ocs_ci/templates/workloads/fio/benchmark_fio.yaml
index 3284fede..2a8a595e 100644
--- a/ocs_ci/templates/workloads/fio/benchmark_fio.yaml
+++ b/ocs_ci/templates/workloads/fio/benchmark_fio.yaml
@@ -12,6 +12,7 @@ spec:
   workload:
     name: "fio_distributed"
     args:
+      image: quay.io/piyushgupta1551/fio-m:ppc64le
       prefill: true
       samples: 3
       servers: 50
diff --git a/ocs_ci/templates/workloads/smallfile/SmallFile.yaml b/ocs_ci/templates/workloads/smallfile/SmallFile.yaml
index 69a490c1..fb1d334f 100644
--- a/ocs_ci/templates/workloads/smallfile/SmallFile.yaml
+++ b/ocs_ci/templates/workloads/smallfile/SmallFile.yaml
@@ -21,6 +21,7 @@ spec:
   workload:
     name: smallfile
     args:
+      image: quay.io/piyushgupta1551/smallfile-m:ppc64le
       clients: 1
       samples: 1
       pause: 50
