diff --git a/ocs_ci/ocs/scale_noobaa_lib.py b/ocs_ci/ocs/scale_noobaa_lib.py
index a417fac9..23298f71 100644
--- a/ocs_ci/ocs/scale_noobaa_lib.py
+++ b/ocs_ci/ocs/scale_noobaa_lib.py
@@ -73,6 +73,11 @@ def check_all_obc_reached_bound_state_in_kube_job(
         job_get_output = kube_job_obj.get(namespace=namespace).get("items")
         if job_get_output is not None and len(job_get_output) == no_of_obc:
             for i in range(no_of_obc):
+                if "status" not in job_get_output[i]:
+                    log.info(f"waiting for OBC status {job_get_output[i]}")
+                    while_iteration_count_1 += 1
+                    time.sleep(3)
+                    continue
                 status = job_get_output[i]["status"]["phase"]
                 log.info(f"obc {job_get_output[i]['metadata']['name']} status {status}")
                 if not status or status != constants.STATUS_BOUND:

