diff --git a/ocs_ci/deployment/deployment.py b/ocs_ci/deployment/deployment.py
index 5cf17f1e..cf519b3d 100644
--- a/ocs_ci/deployment/deployment.py
+++ b/ocs_ci/deployment/deployment.py
@@ -1127,6 +1127,9 @@ def setup_local_storage(storageclass):
     storage_class_device_count = 1
     if platform == constants.AWS_PLATFORM:
         storage_class_device_count = 2
+    if platform == constants.IBM_POWER_PLATFORM:
+        numberofstoragedisks = config.ENV_DATA.get("number_of_storage_disks", 1)
+        storage_class_device_count = numberofstoragedisks
     verify_pvs_created(len(worker_names) * storage_class_device_count)
 
 
