diff --git a/ocs_ci/deployment/helpers/lso_helpers.py b/ocs_ci/deployment/helpers/lso_helpers.py
index 35d60b34..fc36f92a 100644
--- a/ocs_ci/deployment/helpers/lso_helpers.py
+++ b/ocs_ci/deployment/helpers/lso_helpers.py
@@ -232,6 +232,9 @@ def setup_local_storage(storageclass):
     storage_class_device_count = 1
     if platform == constants.AWS_PLATFORM:
         storage_class_device_count = 2
+    if platform == constants.IBM_POWER_PLATFORM:
+        numberofstoragedisks = config.ENV_DATA.get("number_of_storage_disks", 1)
+        storage_class_device_count = numberofstoragedisks
     verify_pvs_created(len(worker_names) * storage_class_device_count)
 
 
