diff --git a/modules/4_nodes/nodes.tf b/modules/4_nodes/nodes.tf
index 07d0106..d805ae2 100644
--- a/modules/4_nodes/nodes.tf
+++ b/modules/4_nodes/nodes.tf
@@ -46,7 +46,7 @@ resource "libvirt_volume" "worker" {
 
 # hostname-ignitions
 data "ignition_file" "b_hostname" {
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
     content {
@@ -57,7 +57,7 @@ EOF
 }
 data "ignition_file" "m_hostname" {
     count       = var.master["count"]
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
     content {
@@ -68,7 +68,7 @@ EOF
 }
 data "ignition_file" "w_hostname" {
     count       = var.worker["count"]
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
 
@@ -82,7 +82,7 @@ EOF
 
 # ignitions
 data "ignition_config" "bootstrap" {
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/bootstrap.ign"
     }
     files = [
@@ -91,7 +91,7 @@ data "ignition_config" "bootstrap" {
 }
 data "ignition_config" "master" {
     count       = var.master["count"]
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/master.ign"
     }
     files       = [
@@ -100,7 +100,7 @@ data "ignition_config" "master" {
 }
 data "ignition_config" "worker" {
     count       = var.worker["count"]
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/worker.ign"
     }
     files       = [
diff --git a/modules/4_nodes/versions.tf b/modules/4_nodes/versions.tf
index ca8afa7..6f80545 100644
--- a/modules/4_nodes/versions.tf
+++ b/modules/4_nodes/versions.tf
@@ -1,8 +1,8 @@
 terraform {
   required_providers {
     ignition = {
-      source  = "community-terraform-providers/ignition"
-      version = "~> 2.1.0"
+      source  = "terraform-providers/ignition"
+      version = "~> 1.2"
     }
     libvirt = {
       source = "dmacvicar/libvirt"
