diff --git a/modules/4_nodes/nodes.tf b/modules/4_nodes/nodes.tf
index 07d0106..c020d93 100644
--- a/modules/4_nodes/nodes.tf
+++ b/modules/4_nodes/nodes.tf
@@ -18,12 +18,19 @@
 #
 ################################################################
 
+locals {
+    bootstrap_ip    = cidrhost(var.network_cidr, 3)
+    first_master_ip = 4
+    first_worker_ip = 11
+}
+
 resource "libvirt_volume" "rhcos_base" {
     name    = "${var.cluster_id}-rhcos-base-vol"
     source  = var.rhcos_image
     pool    = var.storage_pool_name
 }
 
+
 # volumes
 resource "libvirt_volume" "bootstrap" {
     name            = "${var.cluster_id}-bootstrap"
@@ -46,7 +53,7 @@ resource "libvirt_volume" "worker" {
 
 # hostname-ignitions
 data "ignition_file" "b_hostname" {
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
     content {
@@ -57,7 +64,7 @@ EOF
 }
 data "ignition_file" "m_hostname" {
     count       = var.master["count"]
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
     content {
@@ -68,7 +75,7 @@ EOF
 }
 data "ignition_file" "w_hostname" {
     count       = var.worker["count"]
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
 
@@ -82,7 +89,7 @@ EOF
 
 # ignitions
 data "ignition_config" "bootstrap" {
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/bootstrap.ign"
     }
     files = [
@@ -91,7 +98,7 @@ data "ignition_config" "bootstrap" {
 }
 data "ignition_config" "master" {
     count       = var.master["count"]
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/master.ign"
     }
     files       = [
@@ -100,7 +107,7 @@ data "ignition_config" "master" {
 }
 data "ignition_config" "worker" {
     count       = var.worker["count"]
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/worker.ign"
     }
     files       = [
@@ -151,7 +158,7 @@ resource "libvirt_domain" "bootstrap" {
     network_interface {
         network_id  = var.network_id
         hostname    = "bootstrap.${var.cluster_id}.${var.cluster_domain}"
-        mac         = var.bootstrap_mac
+	mac	    = var.bootstrap_mac
     }
 }
 
@@ -176,7 +183,7 @@ resource "libvirt_domain" "master" {
     network_interface {
         network_id  = var.network_id
         hostname    = "master-${count.index}.${var.cluster_id}.${var.cluster_domain}"
-        mac         = var.master_macs[count.index]
+	mac         = var.master_macs[count.index]
     }
 }
 
@@ -204,3 +211,17 @@ resource "libvirt_domain" "worker" {
         mac         = var.worker_macs[count.index]
     }
 }
+
+resource "null_resource" "master_ip" {
+    count       = var.master["count"]
+    triggers    = {
+        address = cidrhost(var.network_cidr, local.first_master_ip + count.index)
+    }
+}
+
+resource "null_resource" "worker_ip" {
+    count       = var.worker["count"]
+    triggers    = {
+        address = cidrhost(var.network_cidr, local.first_worker_ip + count.index)
+    }
+}
diff --git a/modules/4_nodes/versions.tf b/modules/4_nodes/versions.tf
index dadfe22..bbe5216 100644
--- a/modules/4_nodes/versions.tf
+++ b/modules/4_nodes/versions.tf
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     ignition = {
       source = "terraform-providers/ignition"
-      version = "~> 2.1"
+      version = "~> 1.2"
     }
     libvirt = {
       source = "dmacvicar/libvirt"
diff --git a/var.tfvars b/var.tfvars
index 72d8b82..49dedc1 100644
--- a/var.tfvars
+++ b/var.tfvars
@@ -1,15 +1,15 @@
 ### Configure the Libvirt Host values
 libvirt_uri     = "qemu+tcp://localhost/system"
-host_address    = ""
-images_path     = "/home/libvirt/openshift-images"
+host_address    = "<BASTION_IP>"
+images_path     = "<IMAGES_PATH>"
 
 ### Configure the Nodes details
-bastion_image   = "http://remote_server/rhel-8.1-ppc64le-kvm.qcow2"
-rhcos_image     = "http://remote_server/rhcos-4.4.9-ppc64le-qemu.ppc64le.qcow2"
+bastion_image   = "<IMAGES_PATH>/<BASTION_IMAGE>"
+rhcos_image     = "<IMAGES_PATH>/<RHCOS_IMAGE>"
 bastion         = { memory = 8192, vcpu = 2 }
 bootstrap       = { memory = 8192, vcpu = 4, count = 1 }
-master          = { memory = 16384, vcpu = 4, count = 3 }
-worker          = { memory = 16384, vcpu = 4, count = 2 }
+master          = { memory = <MASTER_DESIRED_MEM>, vcpu = <MASTER_DESIRED_CPU>, count = 3 }
+worker          = { memory = <WORKER_DESIRED_MEM>, vcpu = <WORKER_DESIRED_CPU>, count = <WORKERS> }
 cpu_mode        = ""
 network_cidr    = "192.168.88.0/24"
 rhel_username   = "root"
@@ -18,31 +18,31 @@ public_key_file             = "~/.ssh/id_rsa.pub"
 private_key_file            = "~/.ssh/id_rsa"
 private_key                 = ""
 public_key                  = ""
-rhel_subscription_username      = ""
-rhel_subscription_password      = ""
-rhel_subscription_org           = ""
-rhel_subscription_activationkey = ""
+#rhel_subscription_username      = "<RHID_USERNAME>"
+#rhel_subscription_password      = "<RHID_PASSWORD>"
+#rhel_subscription_org           = "<RHID_ORG>"
+#rhel_subscription_activationkey = "<RHID_KEY>"
 
 ### OpenShift variables
-# openshift_install_tarball   = ""
-# openshift_client_tarball    = ""
+openshift_install_tarball   = "https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/<OCP_INSTALLER_SUBPATH>/openshift-install-linux.tar.gz"
+openshift_client_tarball    = "https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/<OCP_INSTALLER_SUBPATH>/openshift-client-linux.tar.gz"
 
-#release_image_override     = ""
+#release_image_override     = "<IMAGE_OVERRIDE>"
 
 pull_secret_file            = "data/pull-secret.txt"
-cluster_domain              = "example.com"
+cluster_domain              = "<CLUSTER_DOMAIN>"
 cluster_id_prefix           = "test"
-cluster_id                  = ""
+cluster_id                  = "ocp<SANITIZED_OCP_VERSION>"
 
 dns_forwarders              = "1.1.1.1; 9.9.9.9"
 installer_log_level         = "info"
-ansible_extra_options       = "-v"
+ansible_extra_options       = "-v --extra-vars '{\"powervm_rmc\":\"false\"}'"
 
 #helpernode_tag             = ""
-#install_playbook_tag       = ""
+install_playbook_tag        = "5df13a33f204c53043c32d583771650c09049d49"
 
-storage_type                = "nfs"
-volume_size                 = "300" # Value in GB
+#storage_type                = "nfs"
+#volume_size                 = "300" # Value in GB
 
 #upgrade_version = ""
 #upgrade_channel = ""  #(stable-4.x, fast-4.x, candidate-4.x) eg. stable-4.5
