diff --git a/modules/1_prepare/prepare.tf b/modules/1_prepare/prepare.tf
index 46c8d8a..01bd45b 100644
--- a/modules/1_prepare/prepare.tf
+++ b/modules/1_prepare/prepare.tf
@@ -157,7 +157,7 @@ EOF
         inline = [
             "#sudo dnf update -y --skip-broken",
             "sudo dnf install -y wget jq git net-tools bind-utils vim python3 python3-devel httpd tar",
-            "sudo dnf install -y openssl-devel libffi-devel"
+            "sudo dnf install -y openssl-devel libffi-devel gcc"
         ]
     }
     provisioner "remote-exec" {
diff --git a/modules/4_nodes/versions.tf b/modules/4_nodes/versions.tf
index ca8afa7..c5195db 100644
--- a/modules/4_nodes/versions.tf
+++ b/modules/4_nodes/versions.tf
@@ -1,7 +1,7 @@
 terraform {
   required_providers {
     ignition = {
-      source  = "community-terraform-providers/ignition"
+      source  = "terraform-providers/ignition"
       version = "~> 2.1.0"
     }
     libvirt = {
