diff --git a/modules/4_nodes/nodes.tf b/modules/4_nodes/nodes.tf
index eeb9d5e..d0393f9 100644
--- a/modules/4_nodes/nodes.tf
+++ b/modules/4_nodes/nodes.tf
@@ -30,14 +30,14 @@ data "ibm_pi_image" "rhcos" {
 
 #bootstrap
 data "ignition_config" "bootstrap" {
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/bootstrap.ign"
     }
     files       = [data.ignition_file.b_hostname.rendered]
 }
 
 data "ignition_file" "b_hostname" {
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
     content {
@@ -71,7 +71,7 @@ resource "ibm_pi_instance" "bootstrap" {
 #master
 data "ignition_config" "master" {
     count       = var.master["count"]
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/master.ign"
     }
     files       = [data.ignition_file.m_hostname[count.index].rendered]
@@ -79,7 +79,7 @@ data "ignition_config" "master" {
 
 data "ignition_file" "m_hostname" {
     count       = var.master["count"]
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
     content {
@@ -124,7 +124,7 @@ resource "ibm_pi_volume" "master" {
 #worker
 data "ignition_file" "w_hostname" {
     count       = var.worker["count"]
-    overwrite   = true
+    filesystem  = "root"
     mode        = "420" // 0644
     path        = "/etc/hostname"
 
@@ -137,7 +137,7 @@ EOF
 
 data "ignition_config" "worker" {
     count       = var.worker["count"]
-    merge {
+    append {
         source  = "http://${var.bastion_ip}:8080/ignition/worker.ign"
     }
     files       = [data.ignition_file.w_hostname[count.index].rendered]
@@ -188,6 +188,7 @@ resource "null_resource" "remove_worker" {
         inline = [<<EOF
 oc adm cordon worker-${count.index}
 oc adm drain worker-${count.index} --force --delete-local-data --ignore-daemonsets
+sleep 30
 oc delete node worker-${count.index}
 EOF
         ]
diff --git a/modules/4_nodes/versions.tf b/modules/4_nodes/versions.tf
index 1f46fd9..87df736 100644
--- a/modules/4_nodes/versions.tf
+++ b/modules/4_nodes/versions.tf
@@ -25,8 +25,8 @@ terraform {
       version = "~> 1.15.0"
     }
     ignition = {
-      source  = "community-terraform-providers/ignition"
-      version = "~> 2.1.0"
+      source  = "terraform-providers/ignition"
+      version = "~> 1.2"
     }
   }
   required_version = "~> 0.13.0"
